<!doctype html>
<html>
	<head>
		<meta charset="utf-8">
	    <meta http-equiv="X-UA-Compatible" content="IE=edge">
	    <meta name="viewport" content="width=device-width, initial-scale=1">
		<title>
     Group C3 [FeFe]-hydrogenases (putative class)
</title>

		<script src="https://ajax.googleapis.com/ajax/libs/jquery/2.1.4/jquery.min.js"></script>

		<script src="http://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/js/bootstrap.min.js"></script>
		<link href="http://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/css/bootstrap.min.css" rel="stylesheet">

		<link href="http://services.birk.au.dk/hyddb/static/common/style.css" rel="stylesheet">

		
    
    <link href="http://services.birc.au.dk/hyddb/static/browser/style.css" rel="stylesheet">

				<div class="container">
			<nav class="navbar navbar-inverse">
			  <div class="container-fluid">
			    <div class="navbar-header">
			      <a class="navbar-brand" href="http://services.birc.au.dk/hyddb/">HydDB</a>
			    </div>

			    <div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
			      <ul class="nav navbar-nav">
			        <li >
			        	<a href="http://services.birc.au.dk/hyddb/">
				        	<span class="glyphicon glyphicon-tags" aria-hidden="true"></span>&nbsp;Classify
			        	</a>
			        </li>
			        <li >
			        	<a href="http://services.birc.au.dk/hyddb/browser/">
				        	<span class="glyphicon glyphicon-th-list" aria-hidden="true"></span>&nbsp;Browse
			        	</a>
			        </li>
			      </ul>
			    </div>
			  </div>
			</nav>
			
<h1>Group C3 [FeFe]-hydrogenases (putative class)</h1>

	<h3>Properties</h3>
	
            <table class="table table-striped">
                <tr>
                    <td class="entry-property-name" width="200">Group</td>
                    <td>[FeFe]-hydrogenases</td>
                </tr>
                <tr>
                    <td class="entry-property-name">Subgroup</td>
                    <td>[FeFe] Group C3: Phosphatase-linked (putative)</td>
                </tr>
                 <tr>
                    <td class="entry-property-name">Function</td>
                    <td>Unconfirmed role.  May serve as H<sub>2</sub>-sensors in regulatory cascades. Enzyme predicted to transfer H<sub>2</sub>-liberated electrons to PAS domain. Associated redox change may modulate activity of associated serine/threonine phosphatase or AAA-type ATPase domains. Members of this class are co-transcribed with fermentative hydrogenases and appear to regulate their transcription. </td>
                </tr>
                 <tr>
                    <td class="entry-property-name">Activity</td>
                    <td>H<sub>2</sub>-sensing?</td>
                </tr>
                 <tr>
                    <td class="entry-property-name">Oxygen Tolerance</td>
                    <td>O<sub>2</sub>-labile</td>
                </tr>
                 <tr>
                    <td class="entry-property-name">Localisation</td>
                    <td>Cytosolic</td>
                </tr>
							</table>
							
							
	<h3>Distribution</h3>
	
            <table class="table table-striped">
                 <tr>
                    <td class="entry-property-name" width="200">Ecosystem Distribution</td>
                    <td>Gastrointestinal tracts, anoxic soils and sediments</td>
                </tr>
                 <tr>
                    <td class="entry-property-name">Taxonomic Distribution</td>
                    <td>Widespread in anaerobic bacteria, including Firmicutes, Deltaproteobacteria, and Spirochaetes </td>
                </tr>
							</table>

							
	<img src="../distribution/fefec3.tif" />
	
	<h3>Genetic Organisation</h3>
	
	<p>Organisation 1: <i>Ruminococcus albus</i> (WP_002849014.1):</p>

	<img src="../organisation/fefec3i.tif" />
	
	<p>Organisation 2: <i>Geobacter sulfurreducens</i> (WP_014552161.1):</p>

	<img src="../organisation/fefec3ii.tif" />
	
	<h3>Architecture</h3>

            <table class="table table-striped">
						    <tr>
                    <td class="entry-property-name" width="200">Structures</td>
                    <td>N/A</td>
                </tr>
                <tr>
                    <td class="entry-property-name" width="200">Number of Subunits</td>
                    <td>1 or 3</td>
                 </tr>										
                <tr>
                    <td class="entry-property-name">Subunit Description</td>
                    <td>Organisation 1: HydS (catalytic subunit with PAS domain), HydP (serine/threonine phosphatase), HydT (thioredoxin)<br />
										Organisation 2: HydS fusion (catalytic subunit with PAS domain fused to AAA-type ATPase domain)<br />
										</td>
                </tr>
                 <tr>
                    <td class="entry-property-name">Catalytic Site</td>
                    <td>[FeFe]-centre</td>
                </tr>
                <tr>
                    <td class="entry-property-name">[FeS] Clusters</td>
                    <td>2 [4Fe4S] clusters</td>
                </tr>
							</table>

	<h3>Literature</h3>


<p><b>Genetics:</b></p>

<ul>
	<li>Calusinska, M., Happe, T., Joris, B., and Wilmotte, A. (2010) The surprising diversity of clostridial hydrogenases: a comparative genomic perspective. <i>Microbiol.</i> <b>156</b>: 1575-1588.</li>
  <li>Greening, C., Biswas, A., Carere, C.R., Jackson, C.J., Taylor, M.C., Stott, M.B., Cook, G.M., and Morales, S.E. (2016) Genomic and metagenomic surveys of hydrogenase distribution indicate H<sub>2</sub> is a widely utilised energy source for microbial growth and survival. <i>ISME J.</i> <b>10</b>: 761-777.</li>
  <li>Wolf, P.G., Biswas, A., Morales, S.E., Greening, C., and Gaskins, H.R. (2016) H<sub>2</sub> metabolism is widespread and diverse among human colonic microbes. <i>Gut Microb.</i>: In press.</li>
  <li>S&oslash;ndergaard, D., Petersen, C.N.S., and Greening, C. (2017) HydDB: a web tool for hydrogenase classification and analysis. Submitted.</li>
	<li>Zheng, Y., Kahnt, J., Kwon, I.H., Mackie, R.I., and Thauer, R.K. (2014) Hydrogen formation and its regulation in <i>Ruminococcus albus</i>: involvement of an electron-bifurcating [FeFe]-hydrogenase, of a non-electron-bifurcating [FeFe]-hydrogenase, and of a putative hydrogen-sensing [FeFe]-hydrogenase. <i>J. Bacteriol.</i> <b>196</b>: 3840-3852.</li>
</ul>
,
<p><b>Physiology:</b></p>

<ul>
	<li>Shaw, A.J., Hogsett, D.A., and Lynd, L.R. (2009) Identification of the [FeFe]-hydrogenase responsible for hydrogen generation in <i>Thermoanaerobacterium saccharolyticum</i> and demonstration of increased ethanol yield via hydrogenase knockout. <i>J. Bacteriol.</i> <b>191</b>: 6457-6464.
</ul>

	<h3>HydDB entries</h3>

	

			<footer class="text-center">
				<small class="text-muted">&copy; 2016 Dan S&oslash;ndergaard &amp; Chris Greening</small>
			</footer>
		</div>
	</body>
</html>

		