<!doctype html>
<html>
	<head>
		<meta charset="utf-8">
	    <meta http-equiv="X-UA-Compatible" content="IE=edge">
	    <meta name="viewport" content="width=device-width, initial-scale=1">
		<title>
     Group 1a [NiFe]-hydrogenases
</title>

		<script src="https://ajax.googleapis.com/ajax/libs/jquery/2.1.4/jquery.min.js"></script>

		<script src="http://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/js/bootstrap.min.js"></script>
		<link href="http://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/css/bootstrap.min.css" rel="stylesheet">

		<link href="http://services.birk.au.dk/hyddb/static/common/style.css" rel="stylesheet">

		
    
    <link href="http://services.birc.au.dk/hyddb/static/browser/style.css" rel="stylesheet">

				<div class="container">
			<nav class="navbar navbar-inverse">
			  <div class="container-fluid">
			    <div class="navbar-header">
			      <a class="navbar-brand" href="http://services.birc.au.dk/hyddb/">HydDB</a>
			    </div>

			    <div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
			      <ul class="nav navbar-nav">
			        <li >
			        	<a href="http://services.birc.au.dk/hyddb/">
				        	<span class="glyphicon glyphicon-tags" aria-hidden="true"></span>&nbsp;Classify
			        	</a>
			        </li>
			        <li >
			        	<a href="http://services.birc.au.dk/hyddb/browser/">
				        	<span class="glyphicon glyphicon-th-list" aria-hidden="true"></span>&nbsp;Browse
			        	</a>
			        </li>
			      </ul>
			    </div>
			  </div>
			</nav>
			
<h1>Group 1a [NiFe]-hydrogenases</h1>

	<h3>Properties</h3>
	
            <table class="table table-striped">
                <tr>
                    <td class="entry-property-name" width="200">Group</td>
                    <td>[NiFe] Group 1: Respiratory H<sub>2</sub>-uptake</sub> [NiFe] hydrogenases</td>
                </tr>
                <tr>
                    <td class="entry-property-name">Subgroup</td>
                    <td>[NiFe] Group 1a: Periplasmic </td>
                </tr>
                 <tr>
                    <td class="entry-property-name">Function</td>
                    <td>Hydrogenotrophic respiration using sulfate, metal, or organohalide compounds as terminal electron acceptors. Enzyme transfers H<sub>2</sub>-liberated electrons through cytochrome <i>c</i><sub>3</sub> to terminal reductase.</td>
                </tr>
                 <tr>
                    <td class="entry-property-name">Activity</td>
                    <td>H<sub>2</sub>-uptake (unidirectional)</td>
                </tr>
                 <tr>
                    <td class="entry-property-name">Oxygen Tolerance</td>
                    <td>O<sub>2</sub>-sensitive (NiFe variants) or O<sub>2</sub>-tolerant (NiFeSe variants)</td>
                </tr>
                 <tr>
                    <td class="entry-property-name">Localisation</td>
                    <td>Periplasmic</td>
                </tr>
							</table>
							
							
	<h3>Distribution</h3>
	
            <table class="table table-striped">
                 <tr>
                    <td class="entry-property-name" width="200">Ecosystem Distribution</td>
                    <td>Anoxic soils and sediments, marine waters, animal gastrointestinal tracts</td>
                </tr>
                 <tr>
                    <td class="entry-property-name">Taxonomic Distribution</td>
                    <td>Widespread in sulfate-reducing Deltaproteobacteria and Firmicutes</td>
                </tr>
							</table>

							
	<img src="../distribution/nife1a.tif" />
	
	<h3>Genetic Organisation</h3>
	
	<p>e.g. <i>Desulfovibrio vulgaris</i> (WP_010939204.1):</p>

	<img src="../organisation/nife1a.tif" />

	<h3>Architecture</h3>

            <table class="table table-striped">
						    <tr>
                    <td class="entry-property-name" width="200">Structures</td>
                    <td>
<a href="http://www.rcsb.org/pdb/explore/explore.do?pdbId=4kn9" target="_blank">4KN9</a> (<i>Desulfomicrobium baculatum</i>, 1.4 &Aring; resolution, active)<br />
<a href="http://www.rcsb.org/pdb/explore/explore.do?pdbId=4ko1" target="_blank">4KO1</a> (<i>Desulfomicrobium baculatum</i>, 1.6 &Aring; resolution, active)<br />
<a href="http://www.rcsb.org/pdb/explore/explore.do?pdbId=4ko2" target="_blank">4KO2</a> (<i>Desulfomicrobium baculatum</i>, 1.6 &Aring; resolution, active)<br />
<a href="http://www.rcsb.org/pdb/explore/explore.do?pdbId=1cc1" target="_blank">1CC1</a> (<i>Desulfomicrobium baculatum</i>, 2.2 &Aring; resolution, active)<br />
<a href="http://www.rcsb.org/pdb/explore/explore.do?pdbId=4ko3" target="_blank">4KO3</a> (<i>Desulfomicrobium baculatum</i>, 1.7 &Aring; resolution, oxidised)<br />
<a href="http://www.rcsb.org/pdb/explore/explore.do?pdbId=4ko4" target="_blank">4KO4</a> (<i>Desulfomicrobium baculatum</i>, 2.0 &Aring; resolution, oxidised)<br />
<a href="http://www.rcsb.org/pdb/explore/explore.do?pdbId=4kl8" target="_blank">4KL8</a> (<i>Desulfomicrobium baculatum</i>, 1.5 &Aring; resolution, oxidised)<br />
<a href="http://www.rcsb.org/pdb/explore/explore.do?pdbId=3zea" target="_blank">3ZEA</a> (<i>Desulfovibrio vulgaris</i>, 1.8 &Aring; resolution, reduced)<br />
<a href="http://www.rcsb.org/pdb/explore/explore.do?pdbId=3ze7" target="_blank">3ZE7</a> (<i>Desulfovibrio vulgaris</i>, 2.0 &Aring; resolution, reduced)<br />
<a href="http://www.rcsb.org/pdb/explore/explore.do?pdbId=3ze8" target="_blank">3ZE8</a> (<i>Desulfovibrio vulgaris</i>, 2.0 &Aring; resolution, reduced)<br />
<a href="http://www.rcsb.org/pdb/explore/explore.do?pdbId=3ze9" target="_blank">3ZE9</a> (<i>Desulfovibrio vulgaris</i>, 1.3 &Aring; resolution, oxidised)<br />
<a href="http://www.rcsb.org/pdb/explore/explore.do?pdbId=3ze6" target="_blank">3ZE6</a> (<i>Desulfovibrio vulgaris</i>, 1.5 &Aring; resolution, oxidised)<br />
<a href="http://www.rcsb.org/pdb/explore/explore.do?pdbId=2wpn" target="_blank">2WPN</a> (<i>Desulfovibrio vulgaris</i>, 2.0 &Aring; resolution, oxidised)<br /></td>
                </tr>
                <tr>
                    <td class="entry-property-name" width="200">Number of Subunits</td>
                    <td>3</td>
                 </tr>										
                <tr>
                    <td class="entry-property-name">Subunit Description</td>
                    <td>HysA (hydrogenase large subunit)<br />
										HysB (hydrogenase small subunit)<br />
										CycA (cytochrome <i>c</i><sub>3</sub> subunit)</td>
                </tr>
                 <tr>
                    <td class="entry-property-name">Catalytic Site</td>
                    <td>[NiFe]-centre or [NiFeSe]-centre</td>
                </tr>
								<tr>
                    <td class="entry-property-name">[FeS] Clusters</td>
                    <td>Proximal: 4Cys[4Fe4S]<br />
										Medial: 4Cys[4Fe4S]<br />
										Distal: 3Cys1His[4Fe4S]</td>
                </tr>
							</table>

	<h3>Literature</h3>


<p><b>Genetics:</b></p>

<ul>
	<li>Calusinska, M., Happe, T., Joris, B., and Wilmotte, A. (2010) The surprising diversity of clostridial hydrogenases: a comparative genomic perspective. <i>Microbiol.</i> <b>156</b>: 1575-1588.</li>
  <li>Greening, C., Biswas, A., Carere, C.R., Jackson, C.J., Taylor, M.C., Stott, M.B., Cook, G.M., and Morales, S.E. (2016) Genomic and metagenomic surveys of hydrogenase distribution indicate H<sub>2</sub> is a widely utilised energy source for microbial growth and survival. <i>ISME J.</i> <b>10</b>: 761-777.</li>
	<li>Menon, N.K., Peck, H.D., Gall, J.L., and Przybyla, A.E. (1987) Cloning and sequencing of the genes encoding the large and small subunits of the periplasmic (NiFeSe) hydrogenase of <i>Desulfovibrio baculatus</i>. <i>J. Bacteriol.</i> <b>169</b>: 5401-5407.</li>
	<li>Valente, F.M., Almeida, C.C., Pacheco, I., Carita, J., Saraiva, L.M., and Pereira, I.A., (2006) Selenium is involved in regulation of periplasmic hydrogenase gene expression in <i>Desulfovibrio vulgaris</i> Hildenborough. <i>J. Bacteriol.</i> <b>188</b>: 3228-3235.</li>
	<li>Voordouw, G., Menon, N.K., LeGall, J., Choi, E.S., Peck, H.D., and Przybyla, A.E. (1989) Analysis and comparison of nucleotide sequences encoding the genes for [NiFe] and [NiFeSe] hydrogenases from <i>Desulfovibrio gigas</i> and <i>Desulfovibrio baculatus</i>. <i>J. Bacteriol.</i> <b>171</b>: 2894-2899.</li>
</ul>

<p><b>Biochemistry:</b></p>
<ul>
	<li>Badziong, W., Thauer, R.K., and Zeikus, J.G. (1978) Isolation and characterization of <i>Desulfovibrio</i> growing on hydrogen plus sulfate as the sole energy source. <i>Arch. Microbiol.</i> <b>116</b>: 41-49.</li>
	<li>Brandis, A. and Thauer, R.K. (1981) Growth of Desulfovibrio species on hydrogen and sulphate as sole energy source. <i>Microbiol.</i> <b>126</b>: 249-252.</li>
	<li>Caffrey, S.M., Park, H.S., Voordouw, J.K., He, Z., Zhou, J., and Voordouw, G. (2007) Function of periplasmic hydrogenases in the sulfate-reducing bacterium <i>Desulfovibrio vulgaris</i> Hildenborough. <i>J. Bacteriol.</i> <b>189</b>: 6159-6167.</li>
	<li>Chardin, B., Giudici-Orticoni, M.T., De Luca, G., Guigliarelli, B. and Bruschi, M. (2003) Hydrogenases in sulfate-reducing bacteria function as chromium reductase. <i>Appl. Microbiol. Biotechnol.</i> <b>63</b>: 315-321.</li>
	<li>Da Silva, S.M., Venceslau, S.S., Fernandes, C.L., Valente, F.M., and Pereira, I.A,. (2008) Hydrogen as an energy source for the human pathogen <i>Bilophila wadsworthia</i>. <i>Antonie Van Leeuwenhoek</i> <b>93</b>: 381-390.</li>
	<li>Li, X., McInerney, M.J., Stahl, D.A. and Krumholz, L.R. (2011) Metabolism of H<sub>2</sub> by <i>Desulfovibrio alaskensis</i> G20 during syntrophic growth on lactate. <i>Microbiol.</i> <b>157</b>: 2912-2921.</li>
	<li>Holliger, C., Hahn, D., Harmsen, H., Ludwig, W., Schumacher, W., Tindall, B., Vazquez, F., Weiss, N., and Zehnder, A.J. (1998) <i>Dehalobacter restrictus</i> gen. nov. and sp. nov., a strictly anaerobic bacterium that reductively dechlorinates tetra- and trichloroethene in an anaerobic respiration. <i>Arch. Microbiol.</i> <b>169</b>: 313-321.</li>
	<li>Malki, S., De Luca, G., Fardeau, M.L., Rousset, M., Belaich, J.P., and Dermoun, Z. (1997) Physiological characteristics and growth behavior of single and double hydrogenase mutants of <i>Desulfovibrio fructosovorans</i>. <i>Arch. Microbiol.</i> <b>167</b>: 38-45.</li>
	<li>Meyer, B., Kuehl, J.V., Deutschbauer, A.M., Arkin, A.P., and Stahl, D.A. (2013) Flexibility of syntrophic enzyme systems in <i>Desulfovibrio</i> species ensures their adaptation capability to environmental changes. <i>J. Bacteriol.</i> <b>195</b>: 4900-4914.</li>
	<li>Nijenhuis, I. and Zinder, S.H. (2005) Characterization of hydrogenase and reductive dehalogenase activities of <i>Dehalococcoides ethenogenes</i> strain 195. <i>Appl. Environ. Microbiol.</i> <b>71</b>: 1664-1667.</li>
	<li>Rohde, M., F&uuml;rstenau, U., Mayer, F., Przybyla, A.E., Peck, H.D., Gall, J., Choi, E.S., and Menon, N.K. (1990) Localization of membrane-associated (NiFe) and (NiFeSe) hydrogenases of <i>Desulfovibrio vulgaris</i> using immunoelectron microscopic procedures. <i>Eur. J. Biochem.</i> <b>191</b>: 389-396.</li>
	<li>Valente, F.M., Pereira, P.M., Venceslau, S.S., Regalla, M., Coelho, A.V., and Pereira, I.A. (2007) The [NiFeSe] hydrogenase from <i>Desulfovibrio vulgaris</i> Hildenborough is a bacterial lipoprotein lacking a typical lipoprotein signal peptide. <i>FEBS Lett.</i> <b>581</b>: 3341-3344.</li>
</ul>

<p><b>Biochemistry:</b>

<ul>
	<li>Baltazar, C.S., Marques, M.C., Soares, C.M., DeLacey, A.M., Pereira, I.A., and Matias, P.M. (2011) Nickelironselenium hydrogenases&mdash;an overview. <i>Eur. J. Inorg. Chem.</i> <b>7</b>: 948-962.</li>
	<li>Berlier, Y., Fauque, G.D., LeGall, J., Choi, E.S., Peck, H.D. and Lespinat, P.A. (1987) Inhibition studies of three classes of <i>Desulfovibrio</i> hydrogenase: application to the further characterization of the multiple hydrogenases found in <i>Desulfovibrio vulgaris</i> Hildenborough. <i>Biochem. Biophys. Res. Commun.</i> <b>2011</b>: 147-153.</li>
	<li>Eidsness, M.K., Scott, R.A., Prickril, B.C., DerVartanian, D.V., Legall, J., Moura, I., Moura, J.J., and Peck, H.D. (1989) Evidence for selenocysteine coordination to the active site nickel in the [NiFeSe] hydrogenases from <i>Desulfovibrio baculatus</i>. <i>Proc. Natl. Acad. Sci. USA</i> <b>86</b>: 147-151.</li>
	<li>Garcin, E., Vernede, X., Hatchikian, E.C., Volbeda, A., Frey, M., and Fontecilla-Camps, J.C. (1999) The crystal structure of a reduced [NiFeSe] hydrogenase provides an image of the activated catalytic center. <i>Structure</i> <b>7</b>: 557-566.</li>
	<li>Guti&eacute;rrez-Sanz, O., Marques, M.C., Baltazar, C.S., Fern&aacute;ndez, V.M., Soares, C.M., Pereira, I.A., and De Lacey, A.L. (2013) Influence of the protein structure surrounding the active site on the catalytic activity of [NiFeSe] hydrogenases. <i>J. Biol. Inorg. Chem.</i> <b>18</b>: 419-427.</li>
	<li>Marques, M.C., Coelho, R., De Lacey, A.L., Pereira, I.A., and Matias, P.M. (2010. The three-dimensional structure of [NiFeSe] hydrogenase from <i>Desulfovibrio vulgaris</i> Hildenborough: a hydrogenase without a bridging ligand in the active site in its oxidised, "as-isolated" state. <i>J. Mol. Biol.</i> <b>396</b>: 893-907.</li>
	<li>Marques, M.C., Coelho, R., Pereira, I.A., and Matias, P.M. (2013) Redox state-dependent changes in the crystal structure of [NiFeSe] hydrogenase from <i>Desulfovibrio vulgaris</i> Hildenborough. <i>Int. J. Hydrogen Energy</i> <b>38</b>: 8664-8682.</li>
	<li>Parkin, A., Goldet, G., Cavazza, C., Fontecilla-Camps, J.C., and Armstrong, F.A. (2008) The difference a Se makes? Oxygen-tolerant hydrogen production by the [NiFeSe]-hydrogenase from <i>Desulfomicrobium baculatum</i>. <i>J. Am. Chem. Soc.</i> <b>130</b>: 13410-13416.</li>
	<li>Teixeira, M., Fauque, G., Moura, I., Lespinat, P.A., Berlier, Y., Prickril, B., Peck, H.D., Xavier, A.V., Gall, J., and Moura, J.J. (1987) Nickel-[iron-sulfur]-selenium-containing hydrogenases from <i>Desulfovibrio baculatus</i> (DSM 1743). <i>Eur. J. Biochem.</i> <b>167</b>: 47-58.</li>
	<li>Valente, F.M., Oliveira, A.S.F., Gnadt, N., Pacheco, I., Coelho, A.V., Xavier, A.V., Teixeira, M., Soares, C.M., and Pereira, I.A. (2005) Hydrogenases in <i>Desulfovibrio vulgaris</i> Hildenborough: structural and physiologic characterisation of the membrane-bound [NiFeSe] hydrogenase. <i>J. Biol. Inorg. Chem.</i> <b>10</b>: 667-682.</li>
	<li>Volbeda, A., Amara, P., Iannello, M., De Lacey, A.L., Cavazza, C. and Fontecilla-Camps, J.C. (2013) Structural foundations for the O<sub>2</sub> resistance of <i>Desulfomicrobium baculatum</i> [NiFeSe]-hydrogenase. <i>Chem. Commun.</i> <b>49</b>: 7061-7063.</li>
</ul>

<p><b>Chemistry:</b></p>

<ul>
	<li>He, S.H., Teixeira, M., LeGall, J., Patil, D.S., Moura, I., Moura, J.J., DerVartanian, D.V., Huynh, B.H., and Peck, H.D. (1989) EPR studies with <sup>77</sup>Se-enriched (NiFeSe) hydrogenase of <i>Desulfovibrio baculatus</i>. Evidence for a selenium ligand to the active site nickel. <i>J. Biol. Chem.</i> <b>264</b>: 2678-2682.</li>
	<li>Reisner, E., Fontecilla-Camps, J.C., and Armstrong, F.A. (2009) Catalytic electrochemistry of a [NiFeSe]-hydrogenase on TiO<sub>2</sub> and demonstration of its suitability for visible-light driven H<sub>2</sub> production. <i>Chem. Commun.</i> <b>5</b>: 550-552.</li>
	<li>Riethausen, J., Rüdiger, O., Gärtner, W., Lubitz, W., and Shafaat, H.S. (2013) Spectroscopic and electrochemical characterization of the [NiFeSe] hydrogenase from <i>Desulfovibrio vulgaris</i> Miyazaki F: Reversible redox behavior and interactions between electron transfer centers. <i>ChemBioChem</i> <b>14</b>: 1714-1719.</li>
	<li>Stein, M. and Lubitz, W. (2001) The electronic structure of the catalytic intermediate Ni-C in [NiFe] and [NiFeSe] hydrogenases. <i>Phys. Chem. Chem. Phys.</i> <b>3</b>: 5115-5120.</li>
	<li>Wang, C.P., Franco, R., Moura, J.J., Moura, I., and Day, E.P. (1992) The nickel site in active <i>Desulfovibrio baculatus</i> [NiFeSe] hydrogenase is diamagnetic. Multifield saturation magnetization measurement of the spin state of Ni(II). <i>J. Biol. Chem.</i> <b>267</b>: 7378-7380.</li>
</ul>

	<h3>HydDB entries</h3>

	

			<footer class="text-center">
				<small class="text-muted">&copy; 2016 Dan S&oslash;ndergaard &amp; Chris Greening</small>
			</footer>
		</div>
	</body>
</html>

		