<!doctype html>
<html>
	<head>
		<meta charset="utf-8">
	    <meta http-equiv="X-UA-Compatible" content="IE=edge">
	    <meta name="viewport" content="width=device-width, initial-scale=1">
		<title>
     Group 1h [NiFe]-hydrogenases
</title>

		<script src="https://ajax.googleapis.com/ajax/libs/jquery/2.1.4/jquery.min.js"></script>

		<script src="http://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/js/bootstrap.min.js"></script>
		<link href="http://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/css/bootstrap.min.css" rel="stylesheet">

		<link href="http://services.birk.au.dk/hyddb/static/common/style.css" rel="stylesheet">

		
    
    <link href="http://services.birc.au.dk/hyddb/static/browser/style.css" rel="stylesheet">

				<div class="container">
			<nav class="navbar navbar-inverse">
			  <div class="container-fluid">
			    <div class="navbar-header">
			      <a class="navbar-brand" href="http://services.birc.au.dk/hyddb/">HydDB</a>
			    </div>

			    <div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
			      <ul class="nav navbar-nav">
			        <li >
			        	<a href="http://services.birc.au.dk/hyddb/">
				        	<span class="glyphicon glyphicon-tags" aria-hidden="true"></span>&nbsp;Classify
			        	</a>
			        </li>
			        <li >
			        	<a href="http://services.birc.au.dk/hyddb/browser/">
				        	<span class="glyphicon glyphicon-th-list" aria-hidden="true"></span>&nbsp;Browse
			        	</a>
			        </li>
			      </ul>
			    </div>
			  </div>
			</nav>
			
<h1>Group 1h [NiFe]-hydrogenases</h1>

	<h3>Properties</h3>
	
            <table class="table table-striped">
                <tr>
                    <td class="entry-property-name" width="200">Group</td>
                    <td>[NiFe] Group 1: Respiratory H<sub>2</sub>-uptake [NiFe] hydrogenases</td>
                </tr>
                <tr>
                    <td class="entry-property-name">Subgroup</td>
                    <td>[NiFe] Group 1h: Actinobacteria-type </td>
                </tr>
                 <tr>
                    <td class="entry-property-name">Function</td>
                    <td>Hydrogenotrophic respiration using O<sub>2</sub> as terminal electron acceptor. Enzyme scavenges electrons from atmospheric H<sub>2</sub> to fuel respiratory chain during carbon-starvation. Route of electron transfer unresolved.</td>
                </tr>
                 <tr>
                    <td class="entry-property-name">Activity</td>
                    <td>H<sub>2</sub>-uptake (unidirectional, high-affinity)</td>
                </tr>
                 <tr>
                    <td class="entry-property-name">Oxygen Tolerance</td>
                    <td>O<sub>2</sub>-tolerant or O<sub>2</sub>-insensitive</td>
                </tr>
                 <tr>
                    <td class="entry-property-name">Localisation</td>
                    <td>Membrane-associated?</td>
                </tr>
							</table>
							
							
	<h3>Distribution</h3>
	
            <table class="table table-striped">
                 <tr>
                    <td class="entry-property-name" width="200">Ecosystem Distribution</td>
                    <td>Upland soils, plant tissues, possibly surface waters</td>
                </tr>
                 <tr>
                    <td class="entry-property-name">Taxonomic Distribution</td>
                    <td>Widespread among obligately aerobic soil bacteria, especially Actinobacteria, Acidobacteria, and Chloroflexi</td>
                </tr>
							</table>

							
	<img src="../distribution/nife1h.tif" />
	
	<h3>Genetic Organisation</h3>
	
	e.g. <i>Mycobacterium smegmatis</i> (WP_003893634.1):</p>

	<img src="../organisation/nife1h.tif" />
								
								
	<h3>Architecture</h3>

            <table class="table table-striped">
						    <tr>
                    <td class="entry-property-name" width="200">Structures</td>
                    <td><a href="http://www.rcsb.org/pdb/explore/explore.do?pdbId=5aa5" target="_blank">5AA5</a> (<i>Ralstonia eutropha</i>, 2.5 &Aring; resolution, active)</td>
                </tr>
                <tr>
                    <td class="entry-property-name" width="200">Number of Subunits</td>
                    <td>3?</td>
                 </tr>								
                 <tr>
                    <td class="entry-property-name">Subunit Description</td>
                    <td>HyhL (hydrogenase large subunit)<br />
										HyhS (hydrogenase small subunit)
										HyhE (putative iron-sulfur protein and proposed physiological electron acceptor)</td>
                 </tr>
                 <tr>
                    <td class="entry-property-name">Catalytic Site</td>
                    <td>[NiFe]-centre</td>
                </tr>
								<tr>
                    <td class="entry-property-name">[FeS] Clusters</td>
                    <td>Proximal: 3Cys1Asp[4Fe4S]<br />
										Medial: 4Cys[4Fe4S]<br />
										Distal: 3Cys1His[4Fe4S]</td>
                </tr>
							</table>
							
	<h3>Important Notes</h3>
	
	<p>The <i>Robiginitalea biformata</i> and <i>Sulfolobus islandicus</i> enzymes are relatively to distantly related to the main group. No studies have yet tested whether these enzymes have a H<sub>2</sub>-scavenging role like other Group 1h [NiFe]-hydrogenases. They may instead represent founding members of a functionally-distinct lineage.</p>
		
	<h3>Literature</h3>

<p><b>Genetics:</b></p>

<ul>
	<li>Berney, M., Greening, C., Hards, K., Collins, D., and Cook, G.M. (2014) Three different [NiFe] hydrogenases confer metabolic flexibility in the obligate aerobe <em>Mycobacterium smegmatis</em>. <em>Environ. Microbiol.</em> <strong>16</strong>: 318-330.</li>
	<li>Constant, P., Chowdhury, S.P., Hesse, L., and Conrad, R. (2011) Co-localization of atmospheric H<sub>2</sub> oxidation activity and high affinity H<sub>2</sub>-oxidizing bacteria in non-axenic soil and sterile soil amended with <i>Streptomyces</i> sp. PCB7. <i>Soil Biol. Biochem.</i> <b>43</b>: 1888-1893.</li>
	<li>Constant, P., Chowdhury, S.P., Hesse, L., Pratscher, J., and Conrad, R. (2011) Genome data mining and soil survey for the novel group 5 [NiFe]-hydrogenase to explore the diversity and ecological importance of presumptive high-affinity H<sub>2</sub>-oxidizing bacteria. <i>Appl. Environ. Microbiol.</i> <b>77</b>: 6027-6035.</li>
  <li>Greening, C., Biswas, A., Carere, C.R., Jackson, C.J., Taylor, M.C., Stott, M.B., Cook, G.M., and Morales, S.E. (2016) Genomic and metagenomic surveys of hydrogenase distribution indicate H<sub>2</sub> is a widely utilised energy source for microbial growth and survival. <i>ISME J.</i> <b>10</b>: 761-777.</li>
	<li>Khdhiri, M., Hesse, L., Popa, M.E., Quiza, L., Lalonde, I., Meredith, L.K., R&ouml;ckmann, T., and Constant, P. (2015) Soil carbon content and relative abundance of high affinity H<sub>2</sub>-oxidizing bacteria predict atmospheric H<sub>2</sub> soil uptake activity better than soil microbial community composition. <i>Soil Biol. Biochem.</i> <b>85</b>: 1-9.</li>
</ul>

<p><b>Physiology:</b></p>	
<ul>
	<li>Berney, M., Greening, C., Conrad, R., Jacobs, W.R., and Cook, G.M. (2014) An obligately aerobic soil bacterium activates fermentative hydrogen production to survive reductive stress during hypoxia. <em>Proc. Natl. Acad. Sci. U. S. A.</em> <strong>111</strong>: 11479-11484.</li>
	<li>Constant, P., Chowdhury, S.P., Pratscher, J., and Conrad, R. (2010) Streptomycetes contributing to atmospheric molecular hydrogen soil uptake are widespread and encode a putative high-affinity [NiFe]-hydrogenase. <em>Environ. Microbiol.</em> <strong>12</strong>: 821-829.</li>
	<li>Constant, P., Poissant, L., and Villemur, R. (2008) Isolation of <em>Streptomyces</em> sp. PCB7, the first microorganism demonstrating high-affinity uptake of tropospheric H<sub>2</sub>. <em>ISME J.</em> <strong>2</strong>: 1066-1076.</li>
	<li>Kanno, M., Constant, P., Tamaki, H., and Kamagata, Y. (2016) Detection and isolation of plant-associated bacteria scavenging atmospheric molecular hydrogen. <em>Environ. Microbiol.</em> 10.1111/1462-2920.13162.</li>
	<li>Greening, C., Berney, M., Hards, K., Cook, G.M., and Conrad, R. (2014) A soil actinobacterium scavenges atmospheric H<sub>2</sub> using two membrane-associated, oxygen-dependent [NiFe] hydrogenases. <em>Proc. Natl. Acad. Sci. U. S. A.</em> <strong>111</strong>: 4257-61.</li>
	<li>Greening, C., Carere, C.R., Rushton-Green, R., Harold, L.K., Hards, K., Taylor, M.C., Morales, S.E., Stott, M.B., and Cook, G.M. (2015) Persistence of the dominant soil phylum <em>Acidobacteria</em> by trace gas scavenging. <em>Proc. Natl. Acad. Sci.</em> <strong>112</strong>: 10497-10502.</li>
	<li>Greening, C., Constant, P., Hards, K., Morales, S.E., Oakeshott, J.G., Russell, R.J., Berney, M., Conrad, R., and Cook, G.M. (2015) Atmospheric hydrogen scavenging: from enzymes to ecosystems. <em>Appl. Environ. Microbiol.</em> <strong>81</strong>: 1190-1199.</li>
	<li>Greening, C., Villas-B&ocirc;as, S.G., Robson, J.R., Berney, M., and Cook, G.M. (2014) The growth and survival of <em>Mycobacterium smegmatis</em> is enhanced by co-metabolism of atmospheric H<sub>2</sub>. <em>PLoS One</em> <strong>9</strong>: e103034.</li>
	<li>Liot, Q. and Constant, P. (2015) Breathing air to save energy - new insights into the ecophysiological role of high-affinity [NiFe]-hydrogenase in <em>Streptomyces avermitilis</em>. <em>Microbiologyopen</em> 10.1002/mbo3.310.</li>
	<li>Meredith, L.K., Rao, D., Bosak, T., Klepac-Ceraj, V., Tada, K.R., Hansel, C.M., et al. (2014) Consumption of atmospheric hydrogen during the life cycle of soil-dwelling actinobacteria. <em>Environ. Microbiol. Rep.</em> <strong>6</strong>: 226-238.</li>
</ul>

<p><b>Biochemistry:</b></p>

<ul>
	<li>Sch&auml;fer, C., Bommer, M., Hennig, S.E., Jeoung, J.-H., Dobbek, H., and Lenz, O. (2016) Structure of an actinobacterial-type [NiFe]-hydrogenase reveals insight into O<sub>2</sub>-tolerant H<sub>2</sub> oxidation. <em>Structure</em> 10.1016/j.str.2015.11.010.</li>
	<li>Sch&auml;fer, C., Friedrich, B., and Lenz, O. (2013) Novel, oxygen-insensitive group 5 [NiFe]-hydrogenase in <em>Ralstonia eutropha</em>. <em>Appl. Environ. Microbiol.</em> <strong>79</strong>: 5137-5145.</li>
</ul>
	
	<h3>HydDB entries</h3>

	

			<footer class="text-center">
				<small class="text-muted">&copy; 2016 Dan S&oslash;ndergaard &amp; Chris Greening</small>
			</footer>
		</div>
	</body>
</html>

		