<!doctype html>
<html>
	<head>
		<meta charset="utf-8">
	    <meta http-equiv="X-UA-Compatible" content="IE=edge">
	    <meta name="viewport" content="width=device-width, initial-scale=1">
		<title>
     Group 1c [NiFe]-hydrogenases
</title>

		<script src="https://ajax.googleapis.com/ajax/libs/jquery/2.1.4/jquery.min.js"></script>

		<script src="http://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/js/bootstrap.min.js"></script>
		<link href="http://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/css/bootstrap.min.css" rel="stylesheet">

		<link href="http://services.birk.au.dk/hyddb/static/common/style.css" rel="stylesheet">

		
    
    <link href="http://services.birc.au.dk/hyddb/static/browser/style.css" rel="stylesheet">

				<div class="container">
			<nav class="navbar navbar-inverse">
			  <div class="container-fluid">
			    <div class="navbar-header">
			      <a class="navbar-brand" href="http://services.birc.au.dk/hyddb/">HydDB</a>
			    </div>

			    <div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
			      <ul class="nav navbar-nav">
			        <li >
			        	<a href="http://services.birc.au.dk/hyddb/">
				        	<span class="glyphicon glyphicon-tags" aria-hidden="true"></span>&nbsp;Classify
			        	</a>
			        </li>
			        <li >
			        	<a href="http://services.birc.au.dk/hyddb/browser/">
				        	<span class="glyphicon glyphicon-th-list" aria-hidden="true"></span>&nbsp;Browse
			        	</a>
			        </li>
			      </ul>
			    </div>
			  </div>
			</nav>
			
<h1>Group 1c [NiFe]-hydrogenases</h1>

	<h3>Properties</h3>
	
            <table class="table table-striped">
                <tr>
                    <td class="entry-property-name" width="200">Group</td>
                    <td>[NiFe] Group 1: Respiratory H<sub>2</sub>-uptake [NiFe] hydrogenases</td>
                </tr>
                <tr>
                    <td class="entry-property-name">Subgroup</td>
                    <td>[NiFe] Group 1c: Hyb-tolerant</td>
                </tr>
                 <tr>
                    <td class="entry-property-name">Function</td>
                    <td>Hydrogenotrophic respiration using fumarate, sulfate, or metals as terminal electron acceptors. Route of electron transfer unresolved. Can also fermentatively evolved H<sub>2</sub> under hypoxia.</td>
                </tr>
                 <tr>
                    <td class="entry-property-name">Activity</td>
                    <td>H<sub>2</sub>-uptake (unidirectional)</td>
                </tr>
                 <tr>
                    <td class="entry-property-name">Oxygen Tolerance</td>
                    <td>O<sub>2</sub>-sensitive</td>
                </tr>
                 <tr>
                    <td class="entry-property-name">Localisation</td>
                    <td>Membrane-bound</td>
                </tr>
							</table>
							
							
	<h3>Distribution</h3>
	
            <table class="table table-striped">
                 <tr>
                    <td class="entry-property-name" width="200">Ecosystem Distribution</td>
                    <td>Animal gastrointestinal tracts, anoxic soils and sediments</td>
                </tr>
                 <tr>
                    <td class="entry-property-name">Taxonomic Distribution</td>
                    <td>Widespread in Enterobacteria and various sulfate-reducing bacteria</td>
                </tr>
							</table>

							
	<img src="../distribution/nife1c.tif" />
	
	<h3>Genetic Organisation</h3>
	
	e.g. <i>Escherichia coli</i> (WP_021568771.1):</p>

	<img src="../organisation/nife1c.tif" />
								
								
	<h3>Architecture</h3>

            <table class="table table-striped">
						    <tr>
                    <td class="entry-property-name" width="200">Structures</td>
                    <td>N/A</td>
                </tr>
                <tr>
                    <td class="entry-property-name" width="200">Number of Subunits</td>
                    <td>4</td>
                 </tr>			
                <tr>
                    <td class="entry-property-name">Subunit Description</td>
                    <td>HybC (hydrogenase large subunit)<br />
										HybO (hydrogenase small subunit)<br />
										HybA (iron-sulfur protein and proposed physiological electron acceptor)<br />
										HybB (transmembrane protein of unresolved function)</td>
                </tr>
                 <tr>
                    <td class="entry-property-name">Catalytic Site</td>
                    <td>[NiFe]-centre</td>
                </tr>
								<tr>
                    <td class="entry-property-name">[FeS] Clusters</td>
                    <td>Proximal: 4Cys[4Fe4S]<br />
										Medial: 3Cys[3Fe4S]<br />
										Distal: 3Cys1His[4Fe4S]</td>
                </tr>
							</table>

	<h3>Literature</h3>

<p><b>Genetics:</b></p>

<ul>
	<li>Coppi, M.V. (2005) The hydrogenases of <i>Geobacter sulfurreducens</i>: a comparative genomic perspective. <i>Microbiol.</i> <b>151</b>: 1239-1254.</li>
  <li>Greening, C., Biswas, A., Carere, C.R., Jackson, C.J., Taylor, M.C., Stott, M.B., Cook, G.M., Morales, S.E. (2016) Genomic and metagenomic surveys of hydrogenase distribution indicate H<sub>2</sub> is a widely utilised energy source for microbial growth and survival. <i>ISME J.</i> <b>10</b>: 761-777.</li>
	<li>Jamieson, D.J., Sawers, R.G., Rugman, P.A., Boxer, D.H. and Higgins, C.F. (1986) Effects of anaerobic regulatory mutations and catabolite repression on regulation of hydrogen metabolism and hydrogenase isoenzyme composition in <i>Salmonella</i> typhimurium. <i>J. Bacteriol.</i> <b>168</b>: 405-411.</li>
	<li>Zbell, A.L., Benoit, S.L., and Maier, R.J. (2007) Differential expression of NiFe uptake-type hydrogenase genes in <i>Salmonella enterica</i> serovar Typhimurium. <i>Microbiol.</i> <b>153</b>: 3508-3516.</li>
	<li>Zbell, A.L., Maier, S.E., and Maier, R.J. (2008) <i>Salmonella enterica</i> serovar Typhimurium NiFe uptake-type hydrogenases are differentially expressed <i>in vivo</i>. <i>Infect. Immun.</i> <b>76</b>: 4445-4454.</li>
	<li>Menon, N.K., Chatelus, C.Y., Dervartanian, M., Wendt, J.C., Shanmugam, K.T., Peck, H.D., and Przybyla, A.E. (1994) Cloning, sequencing, and mutational analysis of the <i>hyb</i> operon encoding <i>Escherichia coli</i> hydrogenase 2. <i>J. Bacteriol.</i> <b>176</b>: 4416-4423.</li>
	<li>Pandelia, M.E., Lubitz, W., and Nitschke, W. (2012) Evolution and diversification of Group 1 [NiFe] hydrogenases. Is there a phylogenetic marker for O<sub>2</sub>-tolerance? <i>BBA Bioenergetics</i> <b>1817</b>: 1565-1575.</li>
	<li>Richard, D.J., Sawers, G., Sargent, F., McWalter, L., and Boxer, D.H. (1999) Transcriptional regulation in response to oxygen and nitrate of the operons encoding the [NiFe] hydrogenases 1 and 2 of <i>Escherichia coli</i>. <i>Microbiol.</i> <b>145</b>: 2903-2912.</li>
	<li>Sargent, F., Ballantine, S.P., Rugman, P.A., Palmer, T. and Boxer, D.H. (1998) Reassignment of the gene encoding the <i>Escherichia coli</i> hydrogenase 2 small subunit. <i>Eur. J. Biochem.</i> <b>255</b>: 746-754.</li>
</ul>

<p><b>Physiology:</b></p>

<ul>
	<li>Ballantine, S.P. and Boxer, D.H. (1985) Nickel-containing hydrogenase isoenzymes from anaerobically grown <i>Escherichia coli</i> K-12. <i>J. Bacteriol.</i> <b>163</b>: 454-459.</li>
	<li>Ban, J., Jiang, W., Li, Y., Zhang, Y., and Li, J. (2010) Functional analysis of hydrogenases and their effects on cell growth and magnetosome synthesis in <i>Magnetospirillum gryphiswaldense</i>. <i>Chinese Sci. Bulletin</i> <b>55</b>: 1271-1277.</li>
	<li>Caccavo, F., Lonergan, D.J., Lovley, D.R., Davis, M., Stolz, J.F., and McInerney, M.J. (1994) <i>Geobacter sulfurreducens</i> sp. nov., a hydrogen-and acetate-oxidizing dissimilatory metal-reducing microorganism. <i>Appl. Environ. Microbiol.</i> <b>60</b>: 3752-3759.</li>
	<li>Coppi, M.V., O'Neil, R.A., and Lovley, D.R. (2004) Identification of an uptake hydrogenase required for hydrogen-dependent reduction of Fe(III) and other electron acceptors by <i>Geobacter sulfurreducens</i>. <i>J. Bacteriol.</i> <b>186</b>: 3022-3028.</li>
	<li>Lamichhane-Khadka, R., Benoit, S.L., Miller-Parks, E.F., and Maier, R.J., 2015. Host hydrogen rather than that produced by the pathogen is important for <i>Salmonella enterica</i> serovar Typhimurium virulence. <i>Infect. Immun.</i> <b>83</b>: 311-316.</li>
	<li>Lamichhane-Khadka, R., Kwiatkowski, A., and Maier, R.J. (2010) The Hyb hydrogenase permits hydrogen-dependent respiratory growth of <i>Salmonella enterica</i> serovar Typhimurium. <i>mBio</i> <b>1</b>: e00284-10.</li>
	<li>Laurinavichene, T.V. and Tsygankov, A.A. (2001) H<sub>2</sub> consumption by <i>Escherichia coli</i> coupled via hydrogenase 1 or hydrogenase 2 to different terminal electron acceptors. <i>FEMS Microbiol. Lett.</i> <b>202</b>: 121-124.</li>
	<li>Laurinavichene, T.V., Zorin, N.A., and Tsygankov, A.A. (2002) Effect of redox potential on activity of hydrogenase 1 and hydrogenase 2 in <i>Escherichia coli</i>. <i>Arch. Microbiol.</i> <b>178</b>: 437-442.</li>
	<li>Maier, L. et al. (2013) Microbiota-derived hydrogen fuels <i>Salmonella typhimurium</i> invasion of the gut ecosystem. <i>Cell Host Microb.</i> <b>14</b>: 641-651.</li>
	<li>Redwood, M.D., Mikheenko, I.P., Sargent, F. and Macaskie, L.E. (2008) Dissecting the roles of <i>Escherichia coli</i> hydrogenases in biohydrogen production. <i>FEMS Microbiol. Lett.</i> <b>278</b>: 48-55.</li>
	<li>Rodrigue, A., Boxer, D.H., Mandrand-Berthelot, M.A., and Wu, L.F. (1996) Requirement for nickel of the transmembrane translocation of NiFe-hydrogenase 2 in <i>Escherichia coli</i>. <i>FEBS Lett.</i> <b>392</b>: 81-86.</li>
	<li>Sawers, R.G., Jamieson, D.J., Higgins, C.F., and Boxer, D.H. (1986) Characterization and physiological roles of membrane-bound hydrogenase isoenzymes from <i>Salmonella typhimurium</i>. <i>J. Bacteriol.</i> <b>168</b>: 398-404.</li>
	<li>Trchounian, K., Soboh, B., Sawers, R.G., and Trchounian, A. (2013) Contribution of hydrogenase 2 to stationary phase H<sub>2</sub> production by <i>Escherichia coli</i> during fermentation of glycerol. <i>Cell Biochem. Biophys.</i> <b>66</b>: 103-108.</li>
	<li>Trchounian, K. and Trchounian, A. (2009) Hydrogenase 2 is most and hydrogenase 1 is less responsible for H<sub>2</sub> production by <i>Escherichia coli</i> under glycerol fermentation at neutral and slightly alkaline pH. <i>Int. J. Hydrogen Energy</i> <b>34</b>: 8839-8845.</li>
	<li>Trchounian, K. and Trchounian, A. (2013) <i>Escherichia coli</i> hydrogenase 4 (<i>hyf</i>) and hydrogenase 2 (<i>hyb</i>) contribution in H<sub>2</sub> production during mixed carbon (glucose and glycerol) fermentation at pH 7.5 and pH 5.5. <i>Int. J. Hydrogen Energy</i> <b>38</b>: 3921-3929.</li>
</ul>

<p><b>Biochemistry:</b></p>

<ul>
	<li>Lukey, M.J., Parkin, A., Roessler, M.M., Murphy, B.J., Harmer, J., Palmer, T., Sargent, F., and Armstrong, F.A. (2010). How <i>Escherichia coli</i> is equipped to oxidize hydrogen under different redox conditions. <i>J. Biol. Chem.</i> <b>285</b>: 3928-3938.</li>
	<li>Muhd Noor, N.D., Nishikawa, K., Nishihara, H., Yoon, K.S., Ogo, S., and Higuchi, Y. (2016) Improved purification, crystallization and crystallographic study of Hyd-2-type [NiFe]-hydrogenase from <i>Citrobacter</i> sp. S-77. <i>Acta Cryst. F:</i> <b>72</b>: 53-58.</li>
</ul>

	<h3>HydDB entries</h3>

	

			<footer class="text-center">
				<small class="text-muted">&copy; 2016 Dan S&oslash;ndergaard &amp; Chris Greening</small>
			</footer>
		</div>
	</body>
</html>

		