<!doctype html>
<html>
	<head>
		<meta charset="utf-8">
	    <meta http-equiv="X-UA-Compatible" content="IE=edge">
	    <meta name="viewport" content="width=device-width, initial-scale=1">
		<title>
     Group 1b [NiFe]-hydrogenases
</title>

		<script src="https://ajax.googleapis.com/ajax/libs/jquery/2.1.4/jquery.min.js"></script>

		<script src="http://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/js/bootstrap.min.js"></script>
		<link href="http://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/css/bootstrap.min.css" rel="stylesheet">

		<link href="http://services.birk.au.dk/hyddb/static/common/style.css" rel="stylesheet">

		
    
    <link href="http://services.birc.au.dk/hyddb/static/browser/style.css" rel="stylesheet">

				<div class="container">
			<nav class="navbar navbar-inverse">
			  <div class="container-fluid">
			    <div class="navbar-header">
			      <a class="navbar-brand" href="http://services.birc.au.dk/hyddb/">HydDB</a>
			    </div>

			    <div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
			      <ul class="nav navbar-nav">
			        <li >
			        	<a href="http://services.birc.au.dk/hyddb/">
				        	<span class="glyphicon glyphicon-tags" aria-hidden="true"></span>&nbsp;Classify
			        	</a>
			        </li>
			        <li >
			        	<a href="http://services.birc.au.dk/hyddb/browser/">
				        	<span class="glyphicon glyphicon-th-list" aria-hidden="true"></span>&nbsp;Browse
			        	</a>
			        </li>
			      </ul>
			    </div>
			  </div>
			</nav>
			
<h1>Group 1b [NiFe]-hydrogenases</h1>

	<h3>Properties</h3>
	
            <table class="table table-striped">
                <tr>
                    <td class="entry-property-name" width="200">Group</td>
                    <td>[NiFe] Group 1: Respiratory H<sub>2</sub>-uptake [NiFe] hydrogenases</td>
                </tr>
                <tr>
                    <td class="entry-property-name">Subgroup</td>
                    <td>[NiFe] Group 1b: Prototypical</td>
                </tr>
                 <tr>
                    <td class="entry-property-name">Function</td>
                    <td>Hydrogenotrophic respiration using sulfate, fumarate, nitrate, metals, and azo compounds as terminal electron acceptors. Enzyme transfers H<sub>2</sub>-liberated electrons through cytochromes to terminal reductase.</td>
                </tr>
                 <tr>
                    <td class="entry-property-name">Activity</td>
                    <td>H<sub>2</sub>-uptake (unidirectional)</td>
                </tr>
                 <tr>
                    <td class="entry-property-name">Oxygen Tolerance</td>
                    <td>O<sub>2</sub>-sensitive</td>
                </tr>
                 <tr>
                    <td class="entry-property-name">Localisation</td>
                    <td>Membrane-bound or periplasmic</td>
                </tr>
							</table>
							
							
	<h3>Distribution</h3>
	
            <table class="table table-striped">
                 <tr>
                    <td class="entry-property-name" width="200">Ecosystem Distribution</td>
                    <td>Anoxic soils and sediments, marine waters, animal gastrointestinal tracts</td>
                </tr>
                 <tr>
                    <td class="entry-property-name">Taxonomic Distribution</td>
                    <td>Widespread in Deltaproteobacteria and Epsilonproteobacteria</td>
                </tr>
							</table>

							
	<img src="../distribution/nife1b.tif" />
	
	<h3>Genetic Organisation</h3>
	
	e.g. <i>Wolinella succinogenes</i> (WP_011139497.1):</p>

	<img src="../organisation/nife1b.tif" />
								
								
	<h3>Architecture</h3>

            <table class="table table-striped">
						    <tr>
                    <td class="entry-property-name" width="200">Structures</td>
                    <td>
<a href="http://www.rcsb.org/pdb/explore/explore.do?pdbId=4u9h" target="_blank">4U9H</a> (<i>Desulfovibrio vulgaris</i>, 0.9 &Aring; resolution, reduced)<br />
<a href="http://www.rcsb.org/pdb/explore/explore.do?pdbId=4u9i" target="_blank">4U9I</a> (<i>Desulfovibrio vulgaris</i>, 1.1 &Aring; resolution, reduced)<br />
<a href="http://www.rcsb.org/pdb/explore/explore.do?pdbId=1h2r" target="_blank">1H2R</a> (<i>Desulfovibrio vulgaris</i>, 1.4 &Aring; resolution, active)<br />
<a href="http://www.rcsb.org/pdb/explore/explore.do?pdbId=1h2a" target="_blank">1H2A</a> (<i>Desulfovibrio vulgaris</i>, 1.8 &Aring; resolution, oxidised)<br />
<a href="http://www.rcsb.org/pdb/explore/explore.do?pdbId=1yrq" target="_blank">1YRQ</a> (<i>Desulfovibrio fructosovorans</i>, 2.1 &Aring; resolution, oxidised)<br />
<a href="http://www.rcsb.org/pdb/explore/explore.do?pdbId=4urh" target="_blank">4URH</a> (<i>Desulfovibrio fructosovorans</i>, 1.4 &Aring; resolution, oxidised)<br />
<a href="http://www.rcsb.org/pdb/explore/explore.do?pdbId=1yqw" target="_blank">1YQW</a> (<i>Desulfovibrio fructosovorans</i>, 1.8 &Aring; resolution, oxidised)<br />
<a href="http://www.rcsb.org/pdb/explore/explore.do?pdbId=4frf" target="_blank">4FRF</a> (<i>Desulfovibrio fructosovorans</i>, 2.7 &Aring; resolution, oxidised)<br />
<a href="http://www.rcsb.org/pdb/explore/explore.do?pdbId=1yq9" target="_blank">1YQ9</a> (<i>Desulfovibrio gigas</i>, 2.4 &Aring; resolution, oxidised)<br />
<a href="http://www.rcsb.org/pdb/explore/explore.do?pdbId=2frv" target="_blank">2FRV</a> (<i>Desulfovibrio gigas</i>, 2.5 &Aring; resolution, oxidised)<br />
<a href="http://www.rcsb.org/pdb/explore/explore.do?pdbId=1frv" target="_blank">1FRV</a> (<i>Desulfovibrio gigas</i>, 2.9 &Aring; resolution, oxidised)<br />
<a href="http://www.rcsb.org/pdb/explore/explore.do?pdbId=1e3d" target="_blank">1E3D</a> (<i>Desulfovibrio desulfuricans</i>, 1.8 &Aring; resolution, oxidised)<br /></td>
                </tr>
                <tr>
                    <td class="entry-property-name" width="200">Number of Subunits</td>
                    <td>3</td>
                 </tr>			
                <tr>
                    <td class="entry-property-name">Subunit Description</td>
                    <td>HynB (hydrogenase large subunit)<br />
										 HynA (hydrogenase small subunit)<br />
										 HynC (cytochrome <i>b</i> subunit)<br />
										 In Deltaproteobacteria, CycA (cytochrome <i>c</i><sub>3</sub> subunit) substitutes for HynC</td>
                </tr>
                 <tr>
                    <td class="entry-property-name">Catalytic Site</td>
                    <td>[NiFe]-centre</td>
                </tr>
								<tr>
                    <td class="entry-property-name">[FeS] Clusters</td>
                    <td>Proximal: 4Cys[4Fe4S] or 3Cys1Asn[4Fe4S]<br />
										Medial: 3Cys[3Fe4S]<br />
										Distal: 3Cys1His[4Fe4S]</td>
                </tr>
							</table>

	<h3>Literature</h3>

<p><b>Genetics:</b></p>

<ul>
	<li>Deckers, H.M., Wilson, F.R., and Voordouw, G. (1990) Cloning and sequencing of a [NiFe] hydrogenase operon from <i>Desulfovibrio vulgaris</i> Miyazaki F. <i>Microbiol.</i> <b>136</b>: 2021-2028.</li>
  <li>Greening, C., Biswas, A., Carere, C.R., Jackson, C.J., Taylor, M.C., Stott, M.B., Cook, G.M., and Morales, S.E. (2016) Genomic and metagenomic surveys of hydrogenase distribution indicate H<sub>2</sub> is a widely utilised energy source for microbial growth and survival. <i>ISME J.</i> <b>10</b>: 761-777.</li>
	<li>Li, C., Peck Jr, H.D., LeGall, J., and Przybyla, A.E. (1987) Cloning, characterization, and sequencing of the genes encoding the large and small subunits of the periplasmic [NiFe] hydrogenase of <i>Desulfovibrio gigas</i>. DNA <b>6</b>: 539-551.</li>
	<li>Rousset, M., Dermoun, Z., Chippaux, M., and Belaich, J.P. (1991) Marker exchange mutagenesis of the <i>hydN</i> genes in <i>Desulfovibrio fructosovorans</i>. <i>Mol. Microbiol.</i> <b>5</b>: 1735-1740.</li>
	<li>Rousset, M., Dermoun, Z., Hatchikian, C.E., and B&eacute;laich, J.P. (1990) Cloning and sequencing of the locus encoding the large and small subunit genes of the periplasmic [NiFe] hydrogenase from <i>Desulfovibrio fructosovorans</i>. <i>Gene</i> <b>94</b>: 95-101.</li>
	<li>Rousset, M., Dermoun, Z., Wall, J.D., and Belaich, J.P. (1993) Analysis of the periplasmic [NiFe] hydrogenase transcription unit from <i>Desulfovibrio fructosovorans</i>. <i>J. Bacteriol.</i> <b>175</b>: 3388-3393.</li>
	<li>Rousset, M., Magro, V., Forget, N., Guigliarelli, B., Belaich, J.P., and Hatchikian, E.C. (1998) Heterologous expression of the <i>Desulfovibrio gigas</i> [NiFe] hydrogenase in <i>Desulfovibrio fructosovorans</i> MR400. <i>J. Bacteriol.</i> <b>180</b>: 4982-4986.</li>
	<li>Voordouw, G. and Brenner, S. (1985) Nucleotide sequence of the gene encoding the hydrogenase from <i>Desulfovibrio vulgaris</i> (Hildenborough). <i>Eur. J. Biochem.</i> <b>148</b>: 515-520.</li>
	<li>Voordouw, G., Menon, N.K., LeGall, J., Choi, E.S., Peck, H.D., and Przybyla, A.E. (1989) Analysis and comparison of nucleotide sequences encoding the genes for [NiFe] and [NiFeSe] hydrogenases from <i>Desulfovibrio gigas</i> and <i>Desulfovibrio baculatus</i>. <i>J. Bacteriol.</i> <b>171</b>: 2894-2899.</li>
	<li>Voordouw, G., Niviere, V., Ferris, F.G., Fedorak, P.M., and Westlake, D.W. (1990) Distribution of hydrogenase genes in <i>Desulfovibrio</i> spp. and their use in identification of species from the oil field environment. <i>Appl. Environ. Microbiol.</i> <b>56</b>: 3748-3754.</li>
	<li>Wawer, C., Jetten, M.S., and Muyzer, G. (1997) Genetic diversity and expression of the [NiFe] hydrogenase large-subunit gene of <i>Desulfovibrio</i> spp. in environmental samples. <i>Appl. Environ. Microbiol.</i> <b>63</b>: 4360-4369.</li>
</ul>

<p><b>Physiology:</b></p>

<ul>
	<li>Badziong, W., Thauer, R.K., and Zeikus, J.G. (1978) Isolation and characterization of <i>Desulfovibrio</i> growing on hydrogen plus sulfate as the sole energy source. <i>Arch. Microbiol.</i> <b>116</b>: 41-49.</li>
	<li>Bell, G.R., LeGall, J., and Peck, H.D. (1974) Evidence for the periplasmic location of hydrogenase in <i>Desulfovibrio gigas</i>. <i>J. Bacteriol.</i> <b>120</b>: 994-997.</li>
	<li>Brandis, A. and Thauer, R.K. (1981) Growth of Desulfovibrio species on hydrogen and sulphate as sole energy source. <i>Microbiol.</i> <b>126</b>: 249-252.</li>
	<li>Caffrey, S.M., Park, H.S., Voordouw, J.K., He, Z., Zhou, J., and Voordouw, G. (2007) Function of periplasmic hydrogenases in the sulfate-reducing bacterium <i>Desulfovibrio vulgaris</i> Hildenborough. <i>J. Bacteriol.</i> <b>189</b>: 6159-6167.</li>
	<li>Casalot, L., Valette, O., De Luca, G., Dermoun, Z., Rousset, M., and de Philip, P., (2002) Construction and physiological studies of hydrogenase depleted mutants of <i>Desulfovibrio fructosovorans</i>. <i>FEMS Microbiol. Lett.</i> <b>214</b>: 107-112.</li>
	<li>Chardin, B., Giudici-Orticoni, M.T., De Luca, G., Guigliarelli, B. and Bruschi, M. (2003) Hydrogenases in sulfate-reducing bacteria function as chromium reductase. <i>Appl. Microbiol. Biotechnol.</i> <b>63</b>: 315-321.</li>
	<li>Da Silva, S.M., Venceslau, S.S., Fernandes, C.L., Valente, F.M., and Pereira, I.A,. (2008) Hydrogen as an energy source for the human pathogen <i>Bilophila wadsworthia</i>. <i>Antonie Van Leeuwenhoek</i> <b>93</b>: 381-390.</li>
	<li>De Luca, G., de Philip, P., Dermoun, Z., Rousset, M., and Verm&eacute;glio, A. (2001) Reduction of technetium(VII) by <i>Desulfovibrio fructosovorans</i> is mediated by the nickel-iron hydrogenase. <i>Appl. Environ. Microbiol.</i> <b>67</b>: 4583-4587.</li>
	<li>Dolla, A., Pohorelic, B.K., Voordouw, J.K., and Voordouw, G. (2000) Deletion of the <i>hmc</i> operon of <i>Desulfovibrio vulgaris</i> subsp. vulgaris Hildenborough hampers hydrogen metabolism and low-redox-potential niche establishment. <i>Arch. Microbiol.</i> <b>174</b>: 143-151.</li>
	<li>Dross, F., Geisler, V., Lenger, R., Theis, F., Krafft, T., Fahrenholz, F., Kojro, E., Duch&ecirc;ne, A. Tripier, D., Juvenal, K. and Kr&ouml;ger, A. (1992) The quinone-reactive Ni/Fe-hydrogenase of <i>Wolinella succinogenes</i>. <i>Eur. J. Biochem.</i> <b>206</b>: 93-102.</li>
	<li>Goodman, T.G. and Hoffman, P.S. (1983) Hydrogenase activity in catalase-positive strains of <i>Campylobacter</i> spp. <i>J. Clin. Microbiol.</i> <b>18</b>: 825-829.</li>
	<li>Gross, R., Simon, J., Theis, F., and Kr&ouml;ger, A. (1998) Two membrane anchors of <i>Wolinella succinogenes</i> hydrogenase and their function in fumarate and polysulfide respiration. <i>Arch. Microbiol.</i> <b>170</b>: 50-58.</li>
	<li>Hansen, M. and Perner, M. (2015) A novel hydrogen oxidizer amidst the sulfur-oxidizing <i>Thiomicrospira</i> lineage. <i>ISME J.</i> <b>9</b>: 696-707.</li>
	<li>Howlett, R.M., Hughes, B.M., Hitchcock, A. and Kelly, D.J. (2012) Hydrogenase activity in the foodborne pathogen <i>Campylobacter jejuni</i> depends upon a novel ABC-type nickel transporter (NikZYXWV) and is SlyD-independent. <i>Microbiol.</i> <b>158</b>: 1645-1655.</li>
	<li>Jankielewicz, A., Klimmek, O., and Kr&ouml;ger, A. (1995) The electron transfer from hydrogenase and formate dehydrogenase to polysulfide reductase in the membrane of <i>Wolinella succinogenes</i>. <i>BBA Bioenerg.</i> <b>1231</b>: 157-162.</li>
	<li>Kuhns, L.G., Benoit, S.L., Bayyareddy, K., Johnson, D., Orlando, R., Evans, A.L., Waldrop, G.L., and Maier, R.J. (2016) Carbon fixation driven by molecular hydrogen results in chemolithoautotrophically enhanced growth of <i>Helicobacter pylori</i>. <i>J. Bacteriol.</i> <b>198</b>: 1423-1428.</li>
	<li>Le Laz, S., Kpebe, A., Lorquin, J., Brugna, M. and Rousset, M. (2014) H<sub>2</sub>-dependent azoreduction by <i>Shewanella oneidensis</i> MR-1: involvement of secreted flavins and both [Ni-Fe] and [Fe-Fe] hydrogenases. <i>Appl. Microbiol. Biotechnol.</i> <b>98</b>: 2699-2707.</li>
	<li>Li, X., McInerney, M.J., Stahl, D.A. and Krumholz, L.R., 2011. Metabolism of H<sub>2</sub> by <i>Desulfovibrio alaskensis</i> G20 during syntrophic growth on lactate. <i>Microbiol.</i> <b>157</b>: 2912-2921.</li>
	<li>Maier, R.J., Fu, C., Gilbert, J., Moshiri, F., Olson, J., and Plaut, A.G. (1996) Hydrogen uptake hydrogenase in <i>Helicobacter pylori</i>. <i>FEMS Microbiol. Lett.</i> <b>141</b>: 71-76.</li>
	<li>Maier, R.J., Olson, J., and Olczak, A. (2003) Hydrogen-oxidizing capabilities of <i>Helicobacter hepaticus</i> and <i>in vivo</i> availability of the substrate. <i>J. Bacteriol.</i> <b>185</b>: 2680-2682.</li>
	<li>Malki, S., De Luca, G., Fardeau, M.L., Rousset, M., Belaich, J.P., and Dermoun, Z. (1997) Physiological characteristics and growth behavior of single and double hydrogenase mutants of <i>Desulfovibrio fructosovorans</i>. <i>Arch. Microbiol.</i> <b>167</b>: 38-45.</li>
	<li>Marshall, M.J. et al. (2008) Hydrogenase-and outer membrane <i>c</i>-type cytochrome-facilitated reduction of technetium(VII) by <i>Shewanella oneidensis</i> MR-1. <i>Environ. Microbiol.</i> <b>10</b>: 125-136.</li>
	<li>Mehta, N.S., Benoit, S., Mysore, J.V., Sousa, R.S., and Maier, R.J. (2005) <i>Helicobacter hepaticus</i> hydrogenase mutants are deficient in hydrogen-supported amino acid uptake and in causing liver lesions in A/J mice. <i>Infect. Immun.</i> <b>73</b>: 5311-5318.</li>
	<li>Meshulam-Simon, G., Behrens, S., Choo, A.D., and Spormann, A.M. (2007) Hydrogen metabolism in <i>Shewanella oneidensis</i> MR-1. <i>Appl. Environ. Microbiol.</i> <b>73</b>: 1153-1165.</li>
	<li>Meyer, B., Kuehl, J.V., Deutschbauer, A.M., Arkin, A.P., and Stahl, D.A. (2013) Flexibility of syntrophic enzyme systems in <i>Desulfovibrio</i> species ensures their adaptation capability to environmental changes. <i>J. Bacteriol.</i> <b>195</b>: 4900-4914.</li>
	<li>Morais-Silva, F.O., Santos, C.I., Rodrigues, R., Pereira, I.A., and Rodrigues-Pousada, C. (2013) Roles of HynAB and Ech, the only two hydrogenases found in the model sulfate reducer <i>Desulfovibrio gigas</i>. <i>J. Bacteriol.</i> <b>195</b>: 4753-4760.</li>
	<li>Olson, J.W. and Maier, R.J. (2002) Molecular hydrogen as an energy source for <i>Helicobacter pylori</i>. <i>Science</i> <b>298</b>: 1788-1790.</li>
	<li>Pereira, I.A., Romão, C.V., Xavier, A.V., LeGall, J., and Teixeira, M. (1998) Electron transfer between hydrogenases and mono-and multiheme cytochromes in <i>Desulfovibrio</i> ssp. <i>J. Biol. Inorg. Chem.</i> <b>3</b> 494-498.</li>
	<li>Rapp-Giles, B.J., Casalot, L., English, R.S., Ringbauer, J.A., Dolla, A., and Wall, J.D. (2000) Cytochrome <i>c</i><sub>3</sub> mutants of <i>Desulfovibrio desulfuricans</i>. <i>Appl. Environ. Microbiol.</i> <b>66</b>: 671-677.</li>
	<li>Schumacher, W., Kroneck, P.M., and Pfennig, N. (1992) Comparative systematic study on "Spirillum" 5175, <i>Campylobacter</i> and <i>Wolinella</i> species. <i>Arch. Microbiol.</i> <b>158</b>: 287-293.</li>
	<li>Weerakoon, D.R., Borden, N.J., Goodson, C.M., Grimes, J., and Olson, J.W. (2009) The role of respiratory donor enzymes in <i>Campylobacter jejuni</i> host colonization and physiology. <i>Microb. Pathog.</i> <b>47</b>: 8-15.</li>
</ul>

<p><b>Biochemistry:</b>

<ul>
	<li>Berlier, Y., Fauque, G.D., LeGall, J., Choi, E.S., Peck, H.D. and Lespinat, P.A. (1987) Inhibition studies of three classes of <i>Desulfovibrio</i> hydrogenase: application to the further characterization of the multiple hydrogenases found in <i>Desulfovibrio vulgaris</i> Hildenborough. <i>Biochem. Biophys. Res. Commun.</i> <b>146</b>: 147-153.</li>
	<li>Dementin, S., Burlat, B., De Lacey, A.L., Pardo, A., Adryanczyk-Perrier, G., Guigliarelli, B., Fernandez, V.M., and Rousset, M. (2004) A glutamate is the essential proton transfer gate during the catalytic cycle of the [NiFe] hydrogenase. <i>J. Biol. Chem.</i> <b>279</b>: 10508-10513.</li>
	<li>Fernandez, V.M., Hatchikian, E.C., and Cammack, R. (1985) Properties and reactivation of two different deactivated forms of <i>Desulfovibrio gigas</i> hydrogenase. <i>BBA Prot. Struct. Mol. Enzymol.</i> <b>832</b>: 69-79.</li>
	<li>Gross, R., Simon, J., Lancaster, C.R.D., and Kr&ouml;ger, A. (1998) Identification of histidine residues in <i>Wolinella succinogenes</i> hydrogenase that are essential for menaquinone reduction by H<sub>2</sub>. <i>Mol. Microbiol.</i> <b>30</b>: 639-646.</li>
	<li>Gross, R., Pisa, R., Sänger, M., Lancaster, C.R.D., and Simon, J. (2004) Characterization of the menaquinone reduction site in the diheme cytochrome <i>b</i> membrane anchor of <i>Wolinella succinogenes</i> NiFe-hydrogenase. <i>J. Biol. Chem.</i> <b>279</b>: 274-281.</li>
	<li>Hatchikian, C.E., Traore, A.S., Fernandez, V.M., and Cammack, R. (1990) Characterization of the nickel-iron periplasmic hydrogenase from <i>Desulfovibrio fructosovorans</i>. <i>Eur. J. Biochem.</i> <b>187</b>: 635-643.</li>
	<li>Higuchi, Y., Ogata, H., Miki, K., Yasuoka, N., and Yagi, T. (1999) Removal of the bridging ligand atom at the Ni-Fe active site of [NiFe] hydrogenase upon reduction with H<sub>2</sub>, as revealed by X-ray structure analysis at 1.4 &Aring; resolution. <i>Structure</i> <b>7</b>: 549-556.</li>
	<li>Higuchi, Y., Yagi, T., and Yasuoka, N. (1997) Unusual ligand structure in Ni-Fe active center and an additional Mg site in hydrogenase revealed by high resolution X-ray structure analysis. <i>Structure</i> <b>5</b>: 1671-1680.</li>
	<li>Leroux, F., Dementin, S., Burlat, B., Cournac, L., Volbeda, A., Champ, S., Martin, L., Guigliarelli, B., Bertrand, P., Fontecilla-Camps, J., and Rousset, M. (2008) Experimental approaches to kinetics of gas diffusion in hydrogenase. <i>Proc. Natl. Acad. Sci. USA</i> <b>105</b>: 11188-11193.</li>
	<li>Matias, P.M., Soares, C.M., Saraiva, L.M., Coelho, R., Morais, J., Le Gall, J., and Carrondo, M.A. (2001) [NiFe] hydrogenase from <i>Desulfovibrio desulfuricans</i> ATCC 27774: gene sequencing, three-dimensional structure determination and refinement at 1.8 &Aring; and modelling studies of its interaction with the tetrahaem cytochrome <i>c</i><sub>3</sub>. <i>J. Biol. Inorg. Chem.</i> <b>6</b>: 63-81.</li>
	<li>Ogata, H., Hirota, S., Nakahara, A., Komori, H., Shibata, N., Kato, T., Kano, K., and Higuchi, Y. (2005) Activation process of [NiFe] hydrogenase elucidated by high-resolution X-ray analyses: conversion of the ready to the unready state. <i>Structure</i> <b>13</b>: 1635-1642.</li>
	<li>Ogata, H., Mizoguchi, Y., Mizuno, N., Miki, K., Adachi, S.I., Yasuoka, N., Yagi, T., Yamauchi, O., Hirota, S., and Higuchi, Y. (2002) Structural studies of the carbon monoxide complex of [NiFe] hydrogenase from <i>Desulfovibrio vulgaris</i> Miyazaki F: suggestion for the initial activation site for dihydrogen. <i>J. Am. Chem. Soc.</i> <b>124</b>: 11628-11635.</li>
	<li>Ogata, H., Nishikawa, K., and Lubitz, W. (2015) Hydrogens detected by subatomic resolution protein crystallography in a [NiFe] hydrogenase. <i>Nature</i> <b>520</b>: 571-574.</li>
	<li>Rousset, M., Montet, Y., Guigliarelli, B., Forget, N., Asso, M., Bertrand, P., Fontecilla-Camps, J.C., and Hatchikian, E.C. (1998) [3Fe-4S] to [4Fe-4S] cluster conversion in <i>Desulfovibrio fructosovorans</i> [NiFe] hydrogenase by site-directed mutagenesis. <i>Proc. Natl. Acad. Sci. USA</i> <b>95</b>: 11625-11630.</li>
	<li>Tatsuhiko, Y., Kimura, K., Daidoji, H., Sakai, F., Tamura, S., and Inokuchi, H. (1976) Properties of purified hydrogenase from the particulate fraction of<i> Desulfovibrio vulgaris</i> Miyazaki. <i>J. Biochem.</i> <b>79</b>: 661-671.</li>
	<li>Vincent, K.A., Parkin, A., Lenz, O., Albracht, S.P., Fontecilla-Camps, J.C., Cammack, R., Friedrich, B., and Armstrong, F.A. (2005) Electrochemical definitions of O<sub>2</sub> sensitivity and oxidative inactivation in hydrogenases. <i>J. Am. Chem. Soc.</i> <b>127</b>: 18179-18189.</li>
	<li>Volbeda, A., Martin, L., Cavazza, C., Matho, M., Faber, B.W., Roseboom, W., Albracht, S.P., Garcin, E., Rousset, M., and Fontecilla-Camps, J.C. (2005) Structural differences between the ready and unready oxidized states of [NiFe] hydrogenases. <i>J. Biol. Inorg. Chem.</i> <b>10</b>: 239-249.</li>
	<li>Volbeda, A., Montet, Y., Vern&egrave;de, X., Hatchikian, E.C., and Fontecilla-Camps, J.C. (2002) High-resolution crystallographic analysis of <i>Desulfovibrio fructosovorans</i> [NiFe] hydrogenase. <i>Int. J. Hydrogen Energy</i> <b>27</b>: 1449-1461.</li>
	<li>Volbeda, A., Charon, M.H., Piras, C., Hatchikian, E.C., Frey, M., and Fontecilla-Camps, J.C. (1995) Crystal structure of the nickel-iron hydrogenase from <i>Desulfovibrio gigas</i>. <i>Nature</i> <b>373</b>: 580-587.</li>
	<li>Yagi, T., Honya, M. and Tamiya, N. (1968) Purification and properties of hydrogenases of different origins. <i>BBA Bioenerget.</i> <b>153</b>: 699-705.</li>
	<li>Yahata, N., Saitoh, T., Takayama, Y., Ozawa, K., Ogata, H., Higuchi, Y., and Akutsu, H. (2006) Redox interaction of cytochrome <i>c</i><sub>3</sub> with [NiFe] hydrogenase from <i>Desulfovibrio vulgaris</i> Miyazaki F. <i>Biochem.</i> <b>45</b>: 1653-1662.</li>
</ul>

<p><b>Chemistry:</b>

<ul>
	<li>Albracht, S.P.J., Kr&ouml;ger, A., Van der Zwaan, J.W., Unden, G., B&ouml;cher, R., Mell, H., and Fontijn, R.D. (1986) Direct evidence for sulphur as a ligand to nickel in hydrogenase: an EPR study of the enzyme from <i>Wolinella succinogenes</i> enriched in <sup>33</sup>S. <i>BBA Prot. Struct. Mol. Enzymol.</i> <b>874</b>: 116-127.</li>
	<li>Amara, P., Volbeda, A., Fontecilla-Camps, J.C., and Field, M.J. (1999) A hybrid density functional theory/molecular mechanics study of nickel-iron hydrogenase: investigation of the active site redox states. <i>J. Am. Chem. Soc. USA</i> <b>121</b>: 4468-4477.</li>
	<li>Bertrand, P., Dole, F., Asso, M., and Guigliarelli, B. (2000) Is there a rate-limiting step in the catalytic cycle of [Ni-Fe] hydrogenases? <i>J. Biol. Inorg. Chem.</i> <b>5</b>: 682-691.</li>
	<li>Cammack, R., Patil, D., Aguirre, R., and Hatchikian, E.C. (1982) Redox properties of the ESR-detectable nickel in hydrogenase from <i>Desulfovibrio gigas</i>. <i>FEBS Lett.</i> <b>142</b>: 289-292.</li>
	<li>Cammack, R., Patil, D.S., Hatchikian, E.C., and Fern&aacute;ndez, V.M. (1987) Nickel and iron-sulphur centres in <i>Desulfovibrio gigas</i> hydrogenase: ESR spectra, redox properties and interactions. <i>BBA Prot. Struct. Mol. Enzymol.</i> <b>912</b>: 98-109.</li>
	<li>Carepo, M., Tierney, D.L., Brondino, C.D., Yang, T.C., Pamplona, A., Telser, J., Moura, I., Moura, J.J., and Hoffman, B.M. (2002) <sup>17</sup>O ENDOR Detection of a solvent-derived Ni-(OH<sub>x</sub>)-Fe bridge that is lost upon activation of the hydrogenase from <i>Desulfovibrio gigas</i>. <i>J. Am. Chem. Soc.</i> <b>124</b>: 281-286.</li>
	<li>de Lacey, A.L., Hatchikian, E.C., Volbeda, A., Frey, M., Fontecilla-Camps, J.C., and Fernandez, V.M. (1997) Infrared-spectroelectrochemical characterization of the [NiFe] hydrogenase of <i>Desulfovibrio gigas</i>. <i>J. Am. Chem. Soc.</i> <b>119</b>: 7181-7189.</li>
	<li>De Lacey, A.L., Fernandez, V.M., Rousset, M., Cavazza, C., and Hatchikian, C.E. (2003) Spectroscopic and kinetic characterization of active site mutants of <i>Desulfovibrio fructosovorans</i> Ni-Fe hydrogenase. <i>J. Biol. Inorg. Chem.</i> <b>8</b>: 129-134.</li>
	<li>De Lacey, A.L., Stadler, C., Fernandez, V.M., Hatchikian, C.E., Fan, H.J., Li, S., and Hall, M.B. (2002) IR spectroelectrochemical study of the binding of carbon monoxide to the active site of Desulfovibrio fructosovorans Ni-Fe hydrogenase. <i>J. Biol. Inorg. Chem.</i> <b>7</b>: 318-326.</li>
	<li>Fichtner, C., Laurich, C., Bothe, E., and Lubitz, W. (2006) Spectroelectrochemical characterization of the [NiFe] hydrogenase of <i>Desulfovibrio vulgaris</i> Miyazaki F. <i>Biochem.</i> <b>45</b>: 9706-9716.</li>
	<li>Fichtner, C., van Gastel, M., and Lubitz, W. (2003) Wavelength dependence of the photo-induced conversion of the Ni-C to the Ni-L redox state in the [NiFe] hydrogenase of <i>Desulfovibrio vulgaris</i> Miyazaki F. <i>Phys. Chem. Chem. Phys.</i> <b>5</b>: 5507-5513.</li>
	<li>Foerster, S., Stein, M., Brecht, M., Ogata, H., Higuchi, Y., and Lubitz, W. (2003) Single crystal EPR studies of the reduced active site of [NiFe] hydrogenase from <i>Desulfovibrio vulgaris</i> Miyazaki F. <i>J. Am. Chem. Soc.</i> <b>125</b>: 83-93.</li>
	<li>Huynh, B.H., Patil, D.S., Moura, I., Teixeira, M., Moura, J.J., DerVartanian, D.V., Czechowski, M.H., Prickril, B.C., Peck, H.D., and LeGall, J. (1987) On the active sites of the [NiFe] hydrogenase from <i>Desulfovibrio gigas</i>. M&ouml;ssbauer and redox-titration studies. <i>J. Biol. Chem.</i> <b>262</b>: 795-800.</li>
	<li>L&eacute;ger, C., Dementin, S., Bertrand, P., Rousset, M., and Guigliarelli, B. (2004) Inhibition and aerobic inactivation kinetics of <i>Desulfovibrio fructosovorans</i> NiFe hydrogenase studied by protein film voltammetry. <i>J. Am. Chem. Soc.</i> <b>126</b>: 12162-12172.</li>
	<li>Li, S. and Hall, M.B. (2001) Predictions of the unready states of [NiFe] <i>Desulfovibrio gigas</i> hydrogenase from density functional theory. <i>Inorg. Chem.</i> <b>40</b>: 18-24.</li>
	<li>Millo, D., Pandelia, M.E., Utesch, T., Wisitruangsakul, N., Mroginski, M.A., Lubitz, W., Hildebrandt, P., and Zebger, I. (2009) Spectroelectrochemical study of the [NiFe] hydrogenase from <i>Desulfovibrio vulgaris</i> Miyazaki F in solution and immobilized on biocompatible gold surfaces. <i>J. Phys. Chem. B</i> <b>113</b>: 15344-15351.</li>
	<li>Niu, S. and Hall, M.B. (2001) The heterolytic bond cleavage of H<sub>2</sub> in the [NiFe] hydrogenase of <i>Desulfovibrio gigas</i> by a nucleophilic addition mechanism. <i>Inorg. Chem.</i> <b>40</b>: 6201-6203.</li>
	<li>Ogata, H. et al (2015) Hydride bridge in [NiFe]-hydrogenase observed by nuclear resonance vibrational spectroscopy. <i>Nat. Commun.</i> <b>6</b>: 7890.</li>
	<li>Pandelia, M.E., Ogata, H., Currell, L.J., Flores, M., and Lubitz, W. (2010) Inhibition of the [NiFe] hydrogenase from <i>Desulfovibrio vulgaris</i> Miyazaki F by carbon monoxide: an FTIR and EPR spectroscopic study. <i>BBA Bioenerg.</i> <b>1797</b>: 304-313.</li>
	<li>Pandelia, M.E., Ogata, H., and Lubitz, W. (2010) Intermediates in the catalytic cycle of [NiFe] hydrogenase: functional spectroscopy of the active site.<i> ChemPhysChem</i> <b>11</b>: 1127-1140.</li>
	<li>Stein, M. and Lubitz, W. (2001) The electronic structure of the catalytic intermediate Ni-C in [NiFe] and [NiFeSe] hydrogenases. <i>Phys. Chem. Chem. Phys.</i> <b>3</b>: 5115-5120.</li>
	<li>Teixeira, M., Moura, I., Xavier, A.V., Moura, J.J., LeGall, J., DerVartanian, D.V., Peck, H.D., and Huynh, B.H. (1989) Redox intermediates of <i>Desulfovibrio gigas</i> [NiFe] hydrogenase generated under hydrogen. M&ouml;ssbauer and EPR characterization of the metal centers. <i>J. Biol. Chem.</i> <b>264</b>: 16435-16450.</li>
	<li>Teixeira, M., Moura, I., Xavier, A.V., Huynh, B.H., DerVartanian, D.V., Peck, H.D., LeGall, J., and Moura, J.J. (1985) Electron paramagnetic resonance studies on the mechanism of activation and the catalytic cycle of the nickel-containing hydrogenase from <i>Desulfovibrio gigas</i>. <i>J. Biol. Chem.</i> <b>260</b>: 8942-8950.</li>
	<li>Trofanchuk, O., Stein, M., Gessner, C., Lendzian, F., Higuchi, Y., and Lubitz, W. (2000) Single crystal EPR studies of the oxidized active site of [NiFe] hydrogenase from <i>Desulfovibrio vulgaris</i> Miyazaki F. <i>J. Biol. Inorg. Chem.</i> <b>5</b>: 36-44.</li>
	<li>van Gastel, M., Stein, M., Brecht, M., Schr&ouml;der, O., Lendzian, F., Bittl, R., Ogata, H., Higuchi, Y., and Lubitz, W. (2006) A single-crystal ENDOR and density functional theory study of the oxidized states of the [NiFe] hydrogenase from <i>Desulfovibrio vulgaris</i> Miyazaki F. <i>J. Biol. Inorg. Chem.</i> <b>11</b>: 41-51.</li>
	<li>Vincent, K.A., Belsey, N.A., Lubitz, W., and Armstrong, F.A. (2006) Rapid and reversible reactions of [NiFe]-hydrogenases with sulfide. <i>J. Am. Chem. Soc. USA</i> <b>128<b>: 7448-7449.</li>
</ul>
	
	<h3>HydDB entries</h3>

	

			<footer class="text-center">
				<small class="text-muted">&copy; 2016 Dan S&oslash;ndergaard &amp; Chris Greening</small>
			</footer>
		</div>
	</body>
</html>

		