<!doctype html>
<html>
	<head>
		<meta charset="utf-8">
	    <meta http-equiv="X-UA-Compatible" content="IE=edge">
	    <meta name="viewport" content="width=device-width, initial-scale=1">
		<title>
     Group 4c [NiFe]-hydrogenases
</title>

		<script src="https://ajax.googleapis.com/ajax/libs/jquery/2.1.4/jquery.min.js"></script>

		<script src="http://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/js/bootstrap.min.js"></script>
		<link href="http://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/css/bootstrap.min.css" rel="stylesheet">

		<link href="http://services.birk.au.dk/hyddb/static/common/style.css" rel="stylesheet">

		
    
    <link href="http://services.birc.au.dk/hyddb/static/browser/style.css" rel="stylesheet">

				<div class="container">
			<nav class="navbar navbar-inverse">
			  <div class="container-fluid">
			    <div class="navbar-header">
			      <a class="navbar-brand" href="http://services.birc.au.dk/hyddb/">HydDB</a>
			    </div>

			    <div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
			      <ul class="nav navbar-nav">
			        <li >
			        	<a href="http://services.birc.au.dk/hyddb/">
				        	<span class="glyphicon glyphicon-tags" aria-hidden="true"></span>&nbsp;Classify
			        	</a>
			        </li>
			        <li >
			        	<a href="http://services.birc.au.dk/hyddb/browser/">
				        	<span class="glyphicon glyphicon-th-list" aria-hidden="true"></span>&nbsp;Browse
			        	</a>
			        </li>
			      </ul>
			    </div>
			  </div>
			</nav>
			
<h1>Group 4c [NiFe]-hydrogenases</h1>

	<h3>Properties</h3>
	
            <table class="table table-striped">
                <tr>
                    <td class="entry-property-name" width="200">Group</td>
                    <td>[NiFe] Group 4: Respiratory H<sub>2</sub>-evolving</sub> [NiFe] hydrogenases</td>
                </tr>
                <tr>
                    <td class="entry-property-name">Subgroup</td>
                    <td>[NiFe] Group 4c: Carbon monoxide-respiring </td>
                </tr>
                 <tr>
                    <td class="entry-property-name">Function</td>
                    <td>Forms respiratory supercomplex that couples CO oxidation at a carbon monoxide dehydrogenase to proton reduction at a CO-tolerant hydrogenase. The energy released is probably used to translocated protons through an unresolved mechanism.  </td>
                </tr>
                 <tr>
                    <td class="entry-property-name">Activity</td>
                    <td>H<sub>2</sub>-evolution</td>
                </tr>
                 <tr>
                    <td class="entry-property-name">Oxygen Tolerance</td>
                    <td>O<sub>2</sub>-tolerant</td>
                </tr>
                 <tr>
                    <td class="entry-property-name">Localisation</td>
                    <td>Membrane-bound</td>
                </tr>
							</table>
							
							
	<h3>Distribution</h3>
	
            <table class="table table-striped">
                 <tr>
                    <td class="entry-property-name" width="200">Ecosystem Distribution</td>
                    <td>In low abundance in anoxic soils</td>
                </tr>
                 <tr>
                    <td class="entry-property-name">Taxonomic Distribution</td>
                    <td>Exclusive to carboxydotrophic Proteobacteria </td>
                </tr>
							</table>

							
	<img src="../distribution/nife4c.tif" />
	
	<h3>Genetic Organisation</h3>
	
	<p>e.g. <i>Carboxydothermus hydrogenoformans</i> (WP_011344721.1):</p>

	<img src="../organisation/nife4c.tif" width="100%" />

	<h3>Architecture</h3>

            <table class="table table-striped">
						    <tr>
                    <td class="entry-property-name" width="200">Structures</td>
                    <td>N/A</td>
                </tr>
                <tr>
                    <td class="entry-property-name" width="200">Number of Subunits</td>
                    <td>7</td>
                 </tr>										
                <tr>
                    <td class="entry-property-name">Subunit Description</td>
                    <td>CooH (hydrogenase large subunit)<br />
										CooL (hydrogenase small subunit)<br />
										CooF (polyferredoxin subunit)<br />
										CooK (NuoH-like transmembrane adaptor)<br />
										CooM (NuoL-like transmembrane adaptor)<br />
										CooX (NuoC-like transmembrane adaptor)<br />
										CooU (conserved hypothetical protein)<br />
										Associates with carbon monoxide dehydrogenase</td>
                </tr>
                 <tr>
                    <td class="entry-property-name">Catalytic Site</td>
                    <td>[NiFe]-centre</td>
                </tr>
                <tr>
                    <td class="entry-property-name">[FeS] Clusters</td>
                    <td>Proximal: 4Cys[4Fe4S]<br />
										Medial: Absent<br />
										Distal: Absent</td>
                </tr>
							</table>

	<h3>Literature</h3>

<p><b>Genetics:</b>

<ul>
	<li>Bonam, D., Lehman, L., Roberts, G.P. and Ludden, P.W. (1989) Regulation of carbon monoxide dehydrogenase and hydrogenase in <i>Rhodospirillum rubrum</i>: effects of CO and oxygen on synthesis and activity. <i>J. Bacteriol.</i> <b>171</b>: 3102-3107.</li>
	<li>Fox, J.D., He, Y., Shelver, D., Roberts, G.P., and Ludden, P.W. (1996) Characterization of the region encoding the CO-induced hydrogenase of <i>Rhodospirillum rubrum</i>. <i>J. Bacteriol.</i> <b>178</b>: 6200-6208.</li>
  <li>Greening, C., Biswas, A., Carere, C.R., Jackson, C.J., Taylor, M.C., Stott, M.B., Cook, G.M., and Morales, S.E. (2016) Genomic and metagenomic surveys of hydrogenase distribution indicate H<sub>2</sub> is a widely utilised energy source for microbial growth and survival. <i>ISME J.</i> <b>10</b>: 761-777.</li>
	<li>Vanzin, G., Yu, J., Smolinski, S., Tek, V., Pennington, G., and Maness, P.C. (2010) Characterization of genes responsible for the CO-linked hydrogen production pathway in <i>Rubrivivax gelatinosus</i>. <i>Appl. Environ. Microbiol.</i> <b>76</b>: 3715-3722.</li>
</ul>

<p><b>Physiology:</b>

<ul>
	<li>Gerhardt, M., Svetlichny, V., Sokolova, T.G., Zavarzin, G.A., and Ringpfeil, M. (1991) Bacterial CO utilization with H<sub>2</sub> production by the strictly anaerobic lithoautotrophic thermophilic bacterium <i>Carboxydothermus hydrogenus</i> DSM 6008 isolated from a hot swamp. <i>FEMS Microbiol. Lett.</i> <b>83</b>: 267-271.</li>
	<li>Kerby, R.L., Ludden, P.W., and Roberts, G.P. (1995) Carbon monoxide-dependent growth of <i>Rhodospirillum rubrum</i>. <i>J. Bacteriol.</i> <b>177</b>: 2241-2244.</li>
	<li>Maness, P.C., Huang, J., Smolinski, S., Tek, V., and Vanzin, G. (2005) Energy generation from the CO oxidation-hydrogen production pathway in <i>Rubrivivax gelatinosus</i>. <i>Appl. Environ. Microbiol.</i> <b>71</b>: 2870-2874.</li>
	<li>Maness, P.C. and Weaver, P.F. (2002) Hydrogen production from a carbon-monoxide oxidation pathway in <i>Rubrivivax gelatinosus</i>. <i>Int. J. Hydrogen Energy</i> <b>27</b>: 1407-1411.</li>
	<li>Maness, P.C., Smolinski, S., Dillon, A.C., Heben, M.J., and Weaver, P.F. (2002) Characterization of the oxygen tolerance of a hydrogenase linked to a carbon monoxide oxidation pathway in <i>Rubrivivax gelatinosus</i>. <i>Appl. Environ. Microbiol.</i> <b>68</b>: 2633-2636.</li>
	<li>Zhao, Y., Cimpoia, R., Liu, Z., and Guiot, S.R. (2011) Kinetics of CO conversion into H<sub>2</sub> by <i>Carboxydothermus hydrogenoformans</i>. <i>Appl. Microbiol. Biotechnol.</i> <b>91</b>: 1677-1684.</li>
</ul>

<p><b>Biochemistry:</b></p>

<ul>
	<li>Soboh, B., Linder, D., and Hedderich, R. (2002) Purification and catalytic properties of a CO-oxidizing: H<sub>2</sub>-evolving enzyme complex from <i>Carboxydothermus hydrogenoformans</i>. <i>Eur. J. Biochem.</i> <b>269</b>: 5712-5721.</li>
	<li>Walker, C.B. et al. (2009) The electron transfer system of syntrophically grown <i>Desulfovibrio vulgaris</i>. <i>J. Bacteriol.</i> <b>191</b>: 5793-5801.</li>
	<li>Singer, S.W., Hirst, M.B., and Ludden, P.W. (2006) CO-dependent H<sub>2</sub> evolution by <i>Rhodospirillum rubrum</i>: role of CODH: CooF complex. <i>BBA Bioenerg.</i> <b>1757</b>: 1582-1591.</li>
</ul>

	<h3>HydDB entries</h3>

	

			<footer class="text-center">
				<small class="text-muted">&copy; 2016 Dan S&oslash;ndergaard &amp; Chris Greening</small>
			</footer>
		</div>
	</body>
</html>

		