<!doctype html>
<html>
	<head>
		<meta charset="utf-8">
	    <meta http-equiv="X-UA-Compatible" content="IE=edge">
	    <meta name="viewport" content="width=device-width, initial-scale=1">
		<title>
     Group 4e [NiFe]-hydrogenases
</title>

		<script src="https://ajax.googleapis.com/ajax/libs/jquery/2.1.4/jquery.min.js"></script>

		<script src="http://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/js/bootstrap.min.js"></script>
		<link href="http://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/css/bootstrap.min.css" rel="stylesheet">

		<link href="http://services.birk.au.dk/hyddb/static/common/style.css" rel="stylesheet">

		
    
    <link href="http://services.birc.au.dk/hyddb/static/browser/style.css" rel="stylesheet">

				<div class="container">
			<nav class="navbar navbar-inverse">
			  <div class="container-fluid">
			    <div class="navbar-header">
			      <a class="navbar-brand" href="http://services.birc.au.dk/hyddb/">HydDB</a>
			    </div>

			    <div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
			      <ul class="nav navbar-nav">
			        <li >
			        	<a href="http://services.birc.au.dk/hyddb/">
				        	<span class="glyphicon glyphicon-tags" aria-hidden="true"></span>&nbsp;Classify
			        	</a>
			        </li>
			        <li >
			        	<a href="http://services.birc.au.dk/hyddb/browser/">
				        	<span class="glyphicon glyphicon-th-list" aria-hidden="true"></span>&nbsp;Browse
			        	</a>
			        </li>
			      </ul>
			    </div>
			  </div>
			</nav>
			
<h1>Group 4e [NiFe]-hydrogenases</h1>

	<h3>Properties</h3>
	
            <table class="table table-striped">
                <tr>
                    <td class="entry-property-name" width="200">Group</td>
                    <td>[NiFe] Group 4: Respiratory H<sub>2</sub>-evolving</sub> [NiFe] hydrogenases</td>
                </tr>
                <tr>
                    <td class="entry-property-name">Subgroup</td>
                    <td>[NiFe] Group 4e: Ferredoxin-coupled, Ech-type </td>
                </tr>
                 <tr>
                    <td class="entry-property-name">Function</td>
                    <td>Forms respiratory complex that couples oxidation of ferredoxin<sub>red</sub> to reduction of protons to H<sub>2</sub> in acetoclastic methanogenesis. This process appears to be coupled to sodium or proton translocation through the transmembrane modules. The complex can also act in the reverse direction during hydrogenotrophic methanogenesis. </td>
                </tr>
                 <tr>
                    <td class="entry-property-name">Activity</td>
                    <td>Bidirectional</td>
                </tr>
                 <tr>
                    <td class="entry-property-name">Oxygen Tolerance</td>
                    <td>O<sub>2</sub>-sensitive</td>
                </tr>
                 <tr>
                    <td class="entry-property-name">Localisation</td>
                    <td>Membrane-bound</td>
                </tr>
							</table>
							
							
	<h3>Distribution</h3>
	
            <table class="table table-striped">
                 <tr>
                    <td class="entry-property-name" width="200">Ecosystem Distribution</td>
                    <td>Anoxic soils and sediments, animal gastrointestinal tracts</td>
                </tr>
                 <tr>
                    <td class="entry-property-name">Taxonomic Distribution</td>
                    <td>Encoded in multiple methanogenic orders and anaerobic bacteria</td>
                </tr>
							</table>

							
	<img src="../distribution/nife4e.tif" />
	
	<h3>Genetic Organisation</h3>
	
	<p>e.g. <i>Methanosarcina barkeri</i> (WP_011305188.1):</p>

	<img src="../organisation/nife4e.tif" />

	<h3>Architecture</h3>

            <table class="table table-striped">
						    <tr>
                    <td class="entry-property-name" width="200">Structures</td>
                    <td>N/A</td>
                </tr>
                <tr>
                    <td class="entry-property-name" width="200">Number of Subunits</td>
                    <td>6</td>
                 </tr>										
                <tr>
                    <td class="entry-property-name">Subunit Description</td>
                    <td>EchA (hydrogenase large subunit)<br />
										EchC (hydrogenase small subunit)<br />
										EchF (iron-sulfur subunit)<br />
										EchB (NuoH-like transmembrane adaptor)<br />
										EchA (NuoL-like transmembrane adaptor)<br />
										EchD (NuoC-like transmembrane adaptor)</td>
                </tr>
                 <tr>
                    <td class="entry-property-name">Catalytic Site</td>
                    <td>[NiFe]-centre</td>
                </tr>
                <tr>
                    <td class="entry-property-name">[FeS] Clusters</td>
                    <td>Proximal: 4Cys[4Fe4S]<br />
										Medial: Absent<br />
										Distal: Absent</td>
                </tr>
							</table>

	<h3>Literature</h3>

<p><b>Genetics:</b></p>

<ul>
	<li>Biswas, R., Zheng, T., Olson, D.G., Lynd, L.R., and Guss, A.M. (2015) Elimination of hydrogenase active site assembly blocks H<sub>2</sub> production and increases ethanol yield in <i>Clostridium thermocellum</i>. <i>Biotechnol. Biofuels</i> <b>8</b>: 20.</li>
	<li>Calusinska, M., Happe, T., Joris, B., and Wilmotte, A. (2010) The surprising diversity of clostridial hydrogenases: a comparative genomic perspective. <i>Microbiol.</i> <b>156</b>: 1575-1588.</li>
	<li>Friedrich, T. and Scheide, D. (2000) The respiratory complex I of bacteria, archaea and eukarya and its module common with membrane-bound multisubunit hydrogenases. <i>FEBS Lett.</i> <b>479</b>: 1-5.</li>
  <li>Greening, C., Biswas, A., Carere, C.R., Jackson, C.J., Taylor, M.C., Stott, M.B., Cook, G.M., and Morales, S.E. (2016) Genomic and metagenomic surveys of hydrogenase distribution indicate H<sub>2</sub> is a widely utilised energy source for microbial growth and survival. <i>ISME J.</i> <b>10</b>: 761-777.</li>
	<li>Hedderich, R. (2004) Energy-converting [NiFe] hydrogenases from archaea and extremophiles: ancestors of complex I. <i>J. Bioenerg. Biomembranes</i> <b>36</b>: 65-75.</li>
	<li>Marreiros, B.C., Batista, A.P., Duarte, A.M., and Pereira, M.M. (2013) A missing link between complex I and group 4 membrane-bound [NiFe] hydrogenases. <i>BBA Bioenerg.</i> <b>1827</b>: 198-209.</li>
	<li>Plugge, C.M., Scholten, J.C., Culley, D.E., Nie, L., Brockman, F.J., and Zhang, W. (2010) Global transcriptomics analysis of the <i>Desulfovibrio vulgaris</i> change from syntrophic growth with Methanosarcina barkeri to sulfidogenic metabolism. <i>Microbiol.</i> <b>156</b>: 2746-2756.</li>
	<li>Rodrigues, R., Valente, F.M., Pereira, I.A., Oliveira, S., and Rodrigues-Pousada, C. (2003) A novel membrane-bound Ech [NiFe] hydrogenase in <i>Desulfovibrio gigas</i>. <i>Biochem. Biophys. Res. Commun.</i> <b>306</b>: 366-375.</li>
</ul>

<p><b>Physiology:</b>

<ul>
	<li>Guss, A.M., Mukhopadhyay, B., Zhang, J.K., and Metcalf, W.W. (2005) Genetic analysis of <i>mch</i> mutants in two <i>Methanosarcina</i> species demonstrates multiple roles for the methanopterin-dependent C-1 oxidation/reduction pathway and differences in H<sub>2</sub> metabolism between closely related species. <i>Mol. Microbiol.</i> <b>55</b>: 1671-1680.</li>
	<li>Hedderich, R. and Forzi, L. (2006) Energy-converting [NiFe] hydrogenases: more than just H<sub>2</sub> activation. <i>J. Mol. Microbiol. Biotechnol.</i> <b>10</b>: 92-104.</li>
	<li>K&uuml;nkel, A., Vorholt, J.A., Thauer, R.K., and Hedderich, R. (1998) An <i>Escherichia coli</i> hydrogenase-3-type hydrogenase in methanogenic archaea. <i>Eur. J. Biochem.</i> <b>252</b>: 467-476.</li>
	<li>Meuer, J., Kuettner, H.C., Zhang, J.K., Hedderich, R., and Metcalf, W.W. (2002) Genetic analysis of the archaeon <i>Methanosarcina barkeri</i> Fusaro reveals a central role for Ech hydrogenase and ferredoxin in methanogenesis and carbon fixation. <i>Proc. Natl. Acad. Sci. USA</i> <b>99</b>: 5632-5637.</li>
	<li>Morais-Silva, F.O., Santos, C.I., Rodrigues, R., Pereira, I.A., and Rodrigues-Pousada, C. (2013) Roles of HynAB and Ech, the only two hydrogenases found in the model sulfate reducer <i>Desulfovibrio gigas</i>. <i>J. Bacteriol.</i> <b>195</b>: 4753-4760.</li>
	<li>Thauer, R.K., 2012. The Wolfe cycle comes full circle. <i>Proc. Acad. Natl. Sci. USA</i> <b>109</b>: 15084-15085.</li>
	<li>Thauer, R.K., Kaster, A.K., Seedorf, H., Buckel, W., and Hedderich, R. (2008) Methanogenic archaea: ecologically relevant differences in energy conservation. <i>Nat. Rev. Microbiol.</i> <b>6</b>: 579-591.</li>
	<li>Welte, C., Kr&auml;tzer, C., and Deppenmeier, U. (2010) Involvement of Ech hydrogenase in energy conservation of <i>Methanosarcina mazei</i>. <i>FEBS J.</i> <b>277</b>:
</ul>

<p><b>Biochemistry:</b></p>

<ul>
	<li>Forzi, L., Koch, J., Guss, A.M., Radosevich, C.G., Metcalf, W.W., and Hedderich, R. (2005) Assignment of the [4Fe-4S] clusters of Ech hydrogenase from <i>Methanosarcina barkeri</i> to individual subunits via the characterization of site-directed mutants. <i>FEBS J.</i> <b>272</b>: 4741-4753.</li>
	<li>Kurkin, S., Meuer, J., Koch, J., Hedderich, R., and Albracht, S.P. (2002) The membrane-bound [NiFe]-hydrogenase (Ech) from <i>Methanosarcina barkeri</i>: unusual properties of the iron-sulphur clusters. <i>Eur. J. Biochem.</i> <b>269</b>: 6101-6111.</li>
	<li>Meuer, J., Bartoschek, S., Koch, J., K&uuml;nkel, A. and Hedderich, R. (1999) Purification and catalytic properties of Ech hydrogenase from <i>Methanosarcina barkeri</i>. <i>Eur. J. Biochem.</i> <b>265</b>: 325-335.</li>
	<li>Soboh, B., Linder, D., and Hedderich, R. (2004) A multisubunit membrane-bound [NiFe] hydrogenase and an NADH-dependent Fe-only hydrogenase in the fermenting bacterium <i>Thermoanaerobacter tengcongensis</i>. <i>Microbiol.</i> <b>150</b>: 2451-2463.</li> 3396-3403.</li>
</ul>

	<h3>HydDB entries</h3>

	

			<footer class="text-center">
				<small class="text-muted">&copy; 2016 Dan S&oslash;ndergaard &amp; Chris Greening</small>
			</footer>
		</div>
	</body>
</html>

		