<!doctype html>
<html>
	<head>
		<meta charset="utf-8">
	    <meta http-equiv="X-UA-Compatible" content="IE=edge">
	    <meta name="viewport" content="width=device-width, initial-scale=1">
		<title>
     Group 4a [NiFe]-hydrogenases
</title>

		<script src="https://ajax.googleapis.com/ajax/libs/jquery/2.1.4/jquery.min.js"></script>

		<script src="http://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/js/bootstrap.min.js"></script>
		<link href="http://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/css/bootstrap.min.css" rel="stylesheet">

		<link href="http://services.birk.au.dk/hyddb/static/common/style.css" rel="stylesheet">

		
    
    <link href="http://services.birc.au.dk/hyddb/static/browser/style.css" rel="stylesheet">

				<div class="container">
			<nav class="navbar navbar-inverse">
			  <div class="container-fluid">
			    <div class="navbar-header">
			      <a class="navbar-brand" href="http://services.birc.au.dk/hyddb/">HydDB</a>
			    </div>

			    <div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
			      <ul class="nav navbar-nav">
			        <li >
			        	<a href="http://services.birc.au.dk/hyddb/">
				        	<span class="glyphicon glyphicon-tags" aria-hidden="true"></span>&nbsp;Classify
			        	</a>
			        </li>
			        <li >
			        	<a href="http://services.birc.au.dk/hyddb/browser/">
				        	<span class="glyphicon glyphicon-th-list" aria-hidden="true"></span>&nbsp;Browse
			        	</a>
			        </li>
			      </ul>
			    </div>
			  </div>
			</nav>
			
<h1>Group 4a [NiFe]-hydrogenases</h1>

	<h3>Properties</h3>
	
            <table class="table table-striped">
                <tr>
                    <td class="entry-property-name" width="200">Group</td>
                    <td>[NiFe] Group 4: Respiratory H<sub>2</sub>-evolving</sub> [NiFe] hydrogenases</td>
                </tr>
                <tr>
                    <td class="entry-property-name">Subgroup</td>
                    <td>[NiFe] Group 4a: Formate hydrogenlyases </td>
                </tr>
                 <tr>
                    <td class="entry-property-name">Function</td>
                    <td>Directly couples formate oxidation to H<sub>2</sub> evolution. It remains unclear whether this process can be coupled to proton-translocation through the transmembrane modules. Reverse reaction occurs <i>in vitro</i> but unresolved whether it occurs <i>in vivo</i>.</td>
                </tr>
                 <tr>
                    <td class="entry-property-name">Activity</td>
                    <td>H<sub>2</sub>-evolution</td>
                </tr>
                 <tr>
                    <td class="entry-property-name">Oxygen Tolerance</td>
                    <td>O<sub>2</sub>-sensitive</td>
                </tr>
                 <tr>
                    <td class="entry-property-name">Localisation</td>
                    <td>Membrane-bound</td>
                </tr>
							</table>
							
							
	<h3>Distribution</h3>
	
            <table class="table table-striped">
                 <tr>
                    <td class="entry-property-name" width="200">Ecosystem Distribution</td>
                    <td>Animal gastrointestinal tracts</td>
                </tr>
                 <tr>
                    <td class="entry-property-name">Taxonomic Distribution</td>
                    <td>Encoded in Enterobacteria and several other Proteobacteria  </td>
                </tr>
							</table>

							
	<img src="../distribution/nife4a.tif" />
	
	<h3>Genetic Organisation</h3>
	
	<p>Organisation 1: <i>Escherichia coli</i> <i>hyc</i> operon (WP_014639275.1):</p>

	<img src="../organisation/nife4a1.tif" width="100%" />

	<p>Organisation 2: <i>Escherichia coli</i> <i>hyf</i> operon (WP_014641051.1):</p>

	<img src="../organisation/nife4a2.tif" width="100%" />
	
	<h3>Architecture</h3>

            <table class="table table-striped">
						    <tr>
                    <td class="entry-property-name" width="200">Structures</td>
                    <td>N/A</td>
                </tr>
                <tr>
                    <td class="entry-property-name" width="200">Number of Subunits</td>
                    <td>6 or 9</td>
                 </tr>										
                <tr>
                    <td class="entry-property-name">Subunit Description</td>
                    <td>HycE / HyfG (hydrogenase large subunit)<br />
										HycG / HyfI (hydrogenase small subunit)<br />
										HycF / HyfH (iron-sulfur protein)<br />
										HycB / HyfA (iron-sulfur protein)<br />
										HycC / HyfB (NuoL-like transmembrane adaptors)<br />
										HycD / HyfC (NuoH-like transmembrane adaptors)<br />
										Hyf-type enzymes additionally associated with three antiporter-like subunits (HyfD, HyfE, HyfF)<br />
										Loosely associated with formate dehydrogenase module (FdhF)</td>
                </tr>
                 <tr>
                    <td class="entry-property-name">Catalytic Site</td>
                    <td>[NiFe]-centre</td>
                </tr>
                <tr>
                    <td class="entry-property-name">[FeS] Clusters</td>
                    <td>Proximal: 4Cys[4Fe4S]<br />
										Medial: Absent<br />
										Distal: Absent</td>
							</table>
	
<h3>Important Notes</h3>

<p>It remains to be resolved whether the formate hydrogenlyase is fermentative (proton-nontranslocating) or respiring (proton-translocating) complex. Based on structural analysis of Complex I suggests, it is possible that the NuoL-like and NuoM-like subunits are proton-translocating. It is likely that Hyf-like complexes can generate membrane potential through its antiporter-like subunits. 

	<h3>Literature</h3>
	
<p><b>Genetics:</b>

<ul>
	<li>Andrews, S.C., Berks, B.C., McClay, J., Ambler, A., Quail, M.A., Golby, P.,	and Guest, J.R. (1997) A 12-cistron Escherichia coli operon (<i>hyf</i>) encoding a putative proton-translocating formate hydrogenlyase system. <i>Microbiol.</i> <b>143</b>: 3633-3647.</li>
	<li>Barrett, E.L., Kwan, H.S., and Macy, J. (1984) Anaerobiosis, formate, nitrate, and <i>pyrA</i> are involved in the regulation of formate hydrogenlyase in <i>Salmonella</i> typhimurium. <i>J. Bacteriol.</i> <b>158</b>: 972-977.</li>
	<li>B&ouml;hm, R., Sauter, M., and B&ouml;ck, A. (1990) Nucleotide sequence and expression of an operon in <i>Escherichia coli</i> coding for formate hydrogenylase components. <i>Mol. Microbiol.</i> <b>4</b>: 231-243.</li>
	<li>Chippaux, M., Casse, F. and Pascal, M.C. (1972) Isolation and phenotypes of mutants from <i>Salmonella</i> typhimurium defective in formate hydrogenlyase activity. <i>J. Bacteriol.</i> <b>110</b>: 766-768.</li>
  <li>Greening, C., Biswas, A., Carere, C.R., Jackson, C.J., Taylor, M.C., Stott, M.B., Cook, G.M., and Morales, S.E. (2016) Genomic and metagenomic surveys of hydrogenase distribution indicate H<sub>2</sub> is a widely utilised energy source for microbial growth and survival. <i>ISME J.</i> <b>10</b>: 761-777.</li>
	<li>Marreiros, B.C., Batista, A.P., Duarte, A.M., and Pereira, M.M. (2013) A missing link between complex I and group 4 membrane-bound [NiFe] hydrogenases. <i>BBA Bioenerg.</i> <b>1827</b>: 198-209.</li>
	<li>Pi, J., Jawed, M., Wang, J., Xu, L., and Yan, Y. (2016) Mutational analysis of the <i>hyc</i>-operon determining the relationship between hydrogenase-3 and NADH pathway in <i>Enterobacter aerogenes</i>. <i>Enzyme Microb. Technol.</i> <b>82</b>: 1-7.</li>
	<li>Sauter, M., B&ouml;hm, R., and B&ouml;ck, A. (1992) Mutational analysis of the operon (<i>hyc</i>) determining hydrogenase 3 formation in <i>Escherichia coli</i>. <i>Mol. Microbiol.</i> <b>6</b>: 1523-1532.</li>
	<li>Sawers, R.G., Ballantine, S.P., and Boxer, D.H. (1985) Differential expression of hydrogenase isoenzymes in <i>Escherichia coli</i> K-12: evidence for a third isoenzyme. <i>J. Bacteriol.</i> <b>164</b>: 1324-1331.</li>
	<li>Schlensog, V. and B&ouml;ck, A. (1990) Identification and sequence analysis of the gene encoding the transcriptional activator of the formate hydrogenlyase system of <i>Escherichia coli</i>. <i>Mol. Microbiol.</i> <b>4</b>: 1319-1327.</li>
	<li>Self, W.T., Hasona, A. and Shanmugam, K.T. (2004) Expression and regulation of a silent operon, <i>hyf</i>, coding for hydrogenase 4 isoenzyme in <i>Escherichia coli</i>. <i>J. Bacteriol.</i> <b>186</b>: 580-587.</li>
	<li>Skibinski, D.A., Golby, P., Chang, Y.S., Sargent, F., Hoffman, R., Harper, R., Guest, J.R., Attwood, M.M., Berks, B.C., and Andrews, S.C. (2002) Regulation of the hydrogenase-4 operon of <i>Escherichia coli</i> by the s54-dependent transcriptional activators FhlA and HyfR. <i>J. Bacteriol.</i> <b>184</b>: 6642-6653.</li>
	<li>Zinoni, F., Birkmann, A., Stadtman, T.C., and B&ouml;ck, A. (1986) Nucleotide sequence and expression of the selenocysteine-containing polypeptide of formate dehydrogenase (formate-hydrogen-lyase-linked) from <i>Escherichia coli</i>. <i>Proc. Acad. Natl. Sci. USA</i> <b>83</b>: 4650-4654.</li>
</ul>

<p><b>Physiology:</b></p>

<ul>
	<li>Lamichhane-Khadka, R., Benoit, S.L., Miller-Parks, E.F., and Maier, R.J. (2015) Host hydrogen rather than that produced by the pathogen is important for <i>Salmonella enterica</i> serovar Typhimurium virulence. <i>Infect. Immun.</i> <b>83</b>: 311-316.</li>
	<li>Mnatsakanyan, N., Bagramyan, K., and Trchounian, A. (2004) Hydrogenase 3 but not hydrogenase 4 is major in hydrogen gas production by <i>Escherichia coli</i> formate hydrogenlyase at acidic pH and in the presence of external formate. <i>Cell Biochem. Biophys.</i> <b>41</b>: 357-365.</li>
	<li>Noguchi, K., Riggins, D.P., Eldahan, K.C., Kitko, R.D., and Slonczewski, J.L. (2010) Hydrogenase-3 contributes to anaerobic acid resistance of <i>Escherichia coli</i>. <i>PLOS One</i> <b>5</b>: e10132.</li>
	<li>Peck Jr, H.D. and Gest, H. (1957) Formic dehydrogenase and the hydrogenlyase enzyme complex in coli-aerogenes bacteria. <i>J. Bacteriol.</i> <b>73</b>: 706-721.</li>
	<li>Redwood, M.D., Mikheenko, I.P., Sargent, F. and Macaskie, L.E. (2008) Dissecting the roles of <i>Escherichia coli</i> hydrogenases in biohydrogen production. <i>FEMS Microbiol. Lett.</i> <b>278</b>: 48-55.</li>
	<li>Qadri, S.H. and Hoare, D.S. (1968) Formic hydrogenlyase and the photoassimilation of formate by a strain of <i>Rhodopseudomonas palustris</i>. <i>J. Bacteriol.</i> <b>95</b>: 2344-2357.</li>
	<li>Sawers, R.G., Jamieson, D.J., Higgins, C.F., and Boxer, D.H. (1986) Characterization and physiological roles of membrane-bound hydrogenase isoenzymes from <i>Salmonella typhimurium</i>. <i>J. Bacteriol.</i> <b>168</b>: 398-404.</li>
	<li>Stephenson, M. and Stickland, L.H. (1932) Hydrogenlyases: Bacterial enzymes liberating molecular hydrogen. <i>Biochemical J.</i> <b>26</b>: 712-724.</li>
	<li>Trchounian, K. and Trchounian, A. (2013) <i>Escherichia coli</i> hydrogenase 4 (<i>hyf</i>) and hydrogenase 2 (<i>hyb</i>) contribution in H<sub>2</sub> production during mixed carbon (glucose and glycerol) fermentation at pH 7.5 and pH 5.5. <i>Int. J. Hydrogen Energy</i> <b>38</b>: 3921-3929.</li>
	<li>Trchounian, K. and Trchounian, A. (2014) Hydrogen producing activity by <i>Escherichia coli</i> hydrogenase 4 (<i>hyf</i>) depends on glucose concentration. <i>Int. J. Hydrogen Energy</i> <b>39</b>: 16914-16918.</li>
	<li>Yoshida, A., Nishimura, T., Kawaguchi, H., Inui, M., and Yukawa, H. (2005) Enhanced hydrogen production from formic acid by formate hydrogen lyase-overexpressing <i>Escherichia coli</i> strains. <i>Appl. Environ. Microbiol.</i> <b>71</b>: 6762-6768.</li>
</ul>

<p><b>Biochemistry:</b></p>

<ul>
	<li>Axley, M.J., Grahame, D.A., and Stadtman, T.C. (1990) <i>Escherichia coli</i> formate-hydrogen lyase. Purification and properties of the selenium-dependent formate dehydrogenase component. <i>J. Biol. Chem.</i> <b>265</b>: 18213-18218.</li>
	<li>Maeda, T., Sanchez-Torres, V., and Wood, T.K. (2007) <i>Escherichia coli</i> hydrogenase 3 is a reversible enzyme possessing hydrogen uptake and synthesis activities. <i>Appl. Microbiol. Biotechnol.</i> <b>76</b>: 1035-1042.</li>
	<li>McDowall, J.S., Hjersing, M.C., Palmer, T., and Sargent, F. (2015) Dissection and engineering of the <i>Escherichia coli</i> formate hydrogenlyase complex. <i>FEBS Lett.</i> <b>589</b>: 3141-3147.</li>
	<li>McDowall, J.S., Murphy, B.J., Haumann, M., Palmer, T., Armstrong, F.A., and Sargent, F. (2014) Bacterial formate hydrogenlyase complex. <i>Proc. Natl. Acad. Sci. USA</i> <b>111</b>: E3948-E3956.</li>
	<li>Pinske, C. and Sargent, F. (2016) Exploring the directionality of <i>Escherichia coli</i> formate hydrogenlyase: a membrane-bound enzyme capable of fixing carbon dioxide to organic acid. <i>MicrobiologyOpen</i>: In press.</li>
</ul>

	<h3>HydDB entries</h3>

	

			<footer class="text-center">
				<small class="text-muted">&copy; 2016 Dan S&oslash;ndergaard &amp; Chris Greening</small>
			</footer>
		</div>
	</body>
</html>

		