<!doctype html>
<html>
	<head>
		<meta charset="utf-8">
	    <meta http-equiv="X-UA-Compatible" content="IE=edge">
	    <meta name="viewport" content="width=device-width, initial-scale=1">
		<title>
     Group 1e [NiFe]-hydrogenases
</title>

		<script src="https://ajax.googleapis.com/ajax/libs/jquery/2.1.4/jquery.min.js"></script>

		<script src="http://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/js/bootstrap.min.js"></script>
		<link href="http://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/css/bootstrap.min.css" rel="stylesheet">

		<link href="http://services.birk.au.dk/hyddb/static/common/style.css" rel="stylesheet">

		
    
    <link href="http://services.birc.au.dk/hyddb/static/browser/style.css" rel="stylesheet">

				<div class="container">
			<nav class="navbar navbar-inverse">
			  <div class="container-fluid">
			    <div class="navbar-header">
			      <a class="navbar-brand" href="http://services.birc.au.dk/hyddb/">HydDB</a>
			    </div>

			    <div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
			      <ul class="nav navbar-nav">
			        <li >
			        	<a href="http://services.birc.au.dk/hyddb/">
				        	<span class="glyphicon glyphicon-tags" aria-hidden="true"></span>&nbsp;Classify
			        	</a>
			        </li>
			        <li >
			        	<a href="http://services.birc.au.dk/hyddb/browser/">
				        	<span class="glyphicon glyphicon-th-list" aria-hidden="true"></span>&nbsp;Browse
			        	</a>
			        </li>
			      </ul>
			    </div>
			  </div>
			</nav>
			
<h1>Group 1e [NiFe]-hydrogenases</h1>

	<h3>Properties</h3>
	
            <table class="table table-striped">
                <tr>
                    <td class="entry-property-name" width="200">Group</td>
                    <td>[NiFe] Group 1: Respiratory H<sub>2</sub>-uptake [NiFe] hydrogenases</td>
                </tr>
                <tr>
                    <td class="entry-property-name">Subgroup</td>
                    <td>[NiFe] Group 1e: Isp-type</td>
                </tr>
                 <tr>
                    <td class="entry-property-name">Function</td>
                    <td>Hydrogenotrophic respiration using sulfur as terminal electron acceptor. Route of electron transfer unresolved. Complex can also evolve H<sub>2</sub> using sulfur-derived electrons possibly through reversed electron flow or electron-bifurcation.</td>
                </tr>
                 <tr>
                    <td class="entry-property-name">Activity</td>
                    <td>Bidirectional (possibly electron-bifurcating)</td>
                </tr>
                 <tr>
                    <td class="entry-property-name">Oxygen Tolerance</td>
                    <td>O<sub>2</sub>-sensitive (some tolerance in microoxic conditions)</td>
                </tr>
                 <tr>
                    <td class="entry-property-name">Localisation</td>
                    <td>Membrane-bound</td>
                </tr>
							</table>
							
							
	<h3>Distribution</h3>
	
            <table class="table table-striped">
                 <tr>
                    <td class="entry-property-name" width="200">Ecosystem Distribution</td>
                    <td>Soil and aquatic systems with high sulfur content</td>
                </tr>
                 <tr>
                    <td class="entry-property-name">Taxonomic Distribution</td>
                    <td>Present in phototrophic and chemotrophic bacteria capable of respiratory sulfur oxidation and reduction</td>
                </tr>
							</table>

							
	<img src="../distribution/nife1e.tif" />
	
	<h3>Genetic Organisation</h3>
	
	e.g. <i>Allochromatium vinosum</i> (WP_012971230.1):</p>

	<img src="../organisation/nife1e.tif" />
								
								
	<h3>Architecture</h3>

            <table class="table table-striped">
						    <tr>
                    <td class="entry-property-name" width="200">Structures</td>
                    <td><a href="http://www.rcsb.org/pdb/explore/explore.do?pdbId=3myr" target="_blank">3MYR</a> (<i>Allochromatium vinosum</i>, 2.1 &Aring; resolution, oxidised, HyiAB only)</td>
                </tr>
                <tr>
                    <td class="entry-property-name" width="200">Number of Subunits</td>
                    <td>4</td>
                 </tr>			
                <tr>
                    <td class="entry-property-name">Subunit Description</td>
                    <td>HyiB (hydrogenase large subunit)<br />
										 HyiA (hydrogenase small subunit)<br />
										 Isp1 (cytochrome <i>b<sup>II</sup></i> subunit)<br />
										 Isp2 (iron-sulfur subunit)</td>
                </tr>
                 <tr>
                    <td class="entry-property-name">Catalytic Site</td>
                    <td>[NiFe]-centre</td>
                </tr>
								<tr>
                    <td class="entry-property-name">[FeS] Clusters</td>
                    <td>Proximal: 4Cys[4Fe4S]<br />
										Medial: 3Cys[3Fe4S]<br />
										Distal: 3Cys1His[4Fe4S]</td>
                </tr>		
							</table>

	<h3>Literature</h3>

<p>Genetics:</b></p>

<ul>

  <li>Greening, C., Biswas, A., Carere, C.R., Jackson, C.J., Taylor, M.C., Stott, M.B., Cook, G.M., Morales, S.E. (2016) Genomic and metagenomic surveys of hydrogenase distribution indicate H<sub>2</sub> is a widely utilised energy source for microbial growth and survival. <i>ISME J.</i> <b>10</b>: 761-777.</li>
	<li>Kov&aacute;cs, &Aacute;, R&aacute;khley, G., Browning, D.F., F&uuml;l&ouml;p, A., Mar&oacute;ti, G., Busby, S.J.W., and Kov&aacute;cs, K.L. (2005) An FNR-type regulator controls the anaerobic expression of Hyn hydrogenase in <i>Thiocapsa roseopersicina</i>. <i>J. Bacteriol.</i> <b>187</b>: 2618-2627.</li>
	<li>Pandelia, M.E., Lubitz, W., and Nitschke, W. (2012) Evolution and diversification of Group 1 [NiFe] hydrogenases. Is there a phylogenetic marker for O<sub>2</sub>-tolerance? <i>BBA Bioenergetics</i> <b>1817</b>: 1565-1575.</li>
	<li>R&aacute;khely, G., Colbeau, A., Garin, J., Vignais, P.M. and Kov&aacute;cs, K.L. (1998) Unusual organization of the genes coding for HydSL, the stable [NiFe] hydrogenase in the photosynthetic bacterium <i>Thiocapsa roseopersicina</i> BBS. <i>J. Bacteriol.</i> <b>180</b>: 1460-1465.</li>
</ul>

<p><b>Physiology:</b>

<ul>
	<li>K&aacute;vacs, K.L., K&aacute;vacs, A.T., Maroti, G., Meszaros, L.S., Balogh, J., Latinovics, D., F&uuml;l&ouml;p, A., David, R., Doroghazi, E., and R&aacute;khely, G. (2005) The hydrogenases of <i>Thiocapsa roseopersicina</i>. <i>Biochem. Soc. Trans.</i> <b>33</b>: 61-63.</li>
	<li>P&aacute;l&aacute;gyi-M&eacute;sz&aacute;ros, L.S., Mar&oacute;ti, J., Latinovics, D., Balogh, T., Klement, E., Medzihradszky, K.F., R&aacute;khely, G., and Kov&aacute;cs, K.L. (2009) Electron-transfer subunits of the NiFe hydrogenases in <i>Thiocapsa roseopersicina</i> BBS. <i>FEBS J.</i> <b>276</b>: 164-174.</li>
	<li>Laurinavichene, T.V., R&aacute;khley, G., Kov&aacute;cs, K.L., and Tsygankov, A.A. (2007) The effect of sulfur compounds on H<sub>2</sub> evolution/consumption reactions, mediated by various hydrogenases, in the purple sulfur bacterium, <i>Thiocapsa roseopersicina</i>. <i>Arch. Microbiol.</i> <b>188</b>: 403-410.</li>
	<li>Teng&ouml;lics, R., M&eacute;sz&aacute;ros, L., Gy&ouml;ri, E., Doffkay, Z., Kov&aacute;cs, K., and R&aacute;khely, G. (2014). Connection between the membrane electron transport system and Hyn hydrogenase in the purple sulfur bacterium, <i>Thiocapsa roseopersicina</i> BBS. <i>BBA Bioenergetics</i> <b>1837</b>: 1691-1698.</li>
	<li>Weissgerber, T., Sylvester, M., Kr&ouml;ninger, L., and Dahl, C. (2014) A comparative quantitative proteomic study identifies new proteins relevant for sulfur oxidation in the purple sulfur bacterium <i>Allochromatium vinosum</i>. <i>Appl. Environ. Microbiol.</i> <b>80</b>: 2279-2292.</li>
</ul>

<p><b>Biochemistry:</b>

<ul>
	<li>Gitlitz, P.H. and Krasna, A.I. (1975) Structural and catalytic properties of hydrogenase from <i>Chromatium</i>. <i>Biochem.</i>: <b>14</b>: 2561-2568.</li>
	<li>Kakuno, T., Kaplan, N.O. and Kamen, M.D., 1977. <i>Chromatium</i> hydrogenase. <i>Proc. Natl. Acad. Sci.</i> USA <b>74:</b> 861-863.</i>
	<li>L&eacute;ger, C., Jones, A.K., Roseboom, W., Albracht, S.P.J., and Armstrong, F.A. (2002) Enzyme electrokinetics: hydrogen evolution and oxidation by <i>Allochromatium vinosum</i> [NiFe]-hydrogenase. <i>Biochem.</i> <b>41</b>: 15736-15746.</li>
	<li>Kellers, P., Ogata, H., and Lubitz, W. (2008) Purification, crystallization and preliminary X-ray analysis of the membrane-bound [NiFe] hydrogenase from <i>Allochromatium vinosum</i>. <i>Acta Crystallogr. F. Struct. Biol. Cryst. Commun.</i> <b>64</b>: 719-722.</li>
	<li>Ogata, H., Kellers, P., and Lubitz, W. (2010) The crystal structure of the [NiFe] hydrogenase from the photosynthetic bacterium <i>Allochromatium vinosum</i>: characterization of the oxidized enzyme (Ni-A state). <i>J. Mol. Biol.</i> <b>402</b>: 428-444.</li>
	<li>Vargas, W.A., Weyman, P.D., Tong, Y.K., Smith, H.O., and Xu, Q. (2011). [NiFe] hydrogenase from <i>Alteromonas macleodii</i> with unusual stability in the presence of oxygen and high temperature. <i>Appl. Environ. Microbiol.</i> <b>77</b>: 1990-1998.</li>
	<li>Weyman, P.D., Vargas, W.A., Tong, Y., Yu, J. Maness, P.-C., Smith, H.O., and Xu, Q. (2011). Heterologous expression of <i>Alteromonas macleodii</i> and <i>Thiocapsa roseopersicina</i> [NiFe]-hydrogenases in <i>Synechococcus elongatus</i>. <i>PLOS One</i> <b>6</b>: e20126.</li>
</ul>

<p><b>Chemistry:</b>

<ul>
	<li>Albracht, S.P.J., Kalkman, M.L., and Slater, E.C., 1983. Magnetic interaction of nickel(III) and the iron-sulphur cluster in hydrogenase from <i>Chromatium vinosum</i>. <i>BBA Bioenergetics</i> <b>724</b>: 309-316.</li>
	<li>Bagley, K.A., Duin, E.C., Roseboom, W., Albracht, S.P., and Woodruff, W.H. (1995) Infrared-detectable group senses changes in charge density on the nickel center in hydrogenase from <i>Chromatium vinosum</i>. <i>Biochem.</i> <b>34</b>: 5527-5535.</li>
	<li>Bagley, K.A., Van Garderen, C.J., Chen, M., Woodruff, W.H., Duin, E.C., and Albracht, S.P. (1994) Infrared studies on the interaction of carbon monoxide with divalent nickel in hydrogenase from Chromatium vinosum. <i>Biochem.</i> <b>33</b>: 9229-9236.</li>
	<li>Bleijlevens, B., Faber, B.W., and Albracht, S.P. (2001) The [NiFe] hydrogenase from <i>Allochromatium vinosum</i> studied in EPR-detectable states: H/D exchange experiments that yield new information about the structure of the active site. <i>J. Biol. Inorg. Chem.</i> <b>6</b>: 763-769.</li>
	<li>Bleijlevens, B., van Broekhuizen, F.A., de Lacey, A.L., Roseboom, W., Fernandez, V.M., and Albracht, S.P.J. (2004) The activation of the [NiFe]-hydrogenase from <i>Allochromatium vinosum</i>. An infrared spectro-electrochemical study. <i>J. Biol. Inorg. Chem.</i> <b>9</b>: 743-752.</i>
	<li>Coremans, J.M.C.C., Vangarderen, C.J., and Albracht, S.P.J. (1992) On the redox equilibrium between H<sub>2</sub> and hydrogenase. <i>Biochim. Biophys. Acta</i> <b>1119</b>: 148-156.</li>
	<li>Coremans, J.M.C.C., Van der Zwaan, J.W. and Albracht, S.P.J. (1992) Distinct redox behaviour of prosthetic groups in ready and unready hydrogenase from <i>Chromatium vinosum</i>. <i>BBA Protein Struct. Mol. Enzymol.</i> <b>1119</b>: 157-168.</li>
	<li>Davidson, G., Choudhury, S.B., Gu, Z., Bose, K., Roseboom, W., Albracht, S.P., and Maroney, M.J. (2000) Structural examination of the nickel site in <i>Chromatium vinosum</i> hydrogenase: redox state oscillations and structural changes accompanying reductive activation and CO binding. <i>Biochem.</i> <b>39</b>: 7468-7479.</li>
	<li>George, S.J., Kurkin, S., Thorneley, R.N.F., and Albracht, S.P.J. (2004) Reactions of H<sub>2</sub>, CO, and O<sub>2</sub> with active [NiFe]-hydrogenase from <i>Allochromatium vinosum</i>. A stopped-flow infrared study. <i>Biochem.</i> <b>43</b>: 6808-6819.</li>
	<li>Happe, R.P., Roseboom, W., and Albracht, S.P., (1999). Pre-steady-state kinetics of the reactions of [NiFe]-hydrogenase from <i>Chromatium vinosum</i> with H<sub>2</sub> and CO. <i>Eur. J. Biochem.</i>, <b>259</b>: 602-608.</li>
	<li>Jones, A.K., Lamle, S.E., Pershad, H.R., Vincent, K.A., Albracht, S.P.J., and Armstrong, F.A. (2003) Enzyme electrokinetics: Electrochemical studies of the anaerobic interconversions between active and inactive states of <i>Allochromatium vinosum</i> [NiFe]-hydrogenase. <i>J. Am. Chem. Soc.</i> <b>125</b>: 8505-8514.</li>
	<li>Kurkin, S., George, S.J., Thorneley, R.N.F., and Albracht, S.P.J. (2004). Hydrogen-induced activation of the [NiFe]-hydrogenase from <i>Allochromatium vinosum</i> as studied by stopped-flow infrared spectroscopy. <i>Biochem.</i> <b>43</b>: 6820-6831.</li>
	<li>Lamle, S.E., Albracht, S.P.J., and Armstrong, F.A. (2004) Electrochemical potential-step investigations of the aerobic interconversions of [NiFe]-hydrogenase from <i>Allochromatium vinosum</i>: insights into the puzzling difference between unready and ready oxidized inactive states. <i>J. Am. Chem. Soc.</i> <b>126</b>: 14899-14909.</li>
	<li>Pershad, H.R., Duff, J.L., Heering, H.A., Duin, E.C., Albracht, S.P., and Armstrong, F.A. (1999) Catalytic electron transport in <i>Chromatium vinosum</i> [NiFe]-hydrogenase: application of voltammetry in detecting redox-active centers and establishing that hydrogen oxidation is very fast even at potentials close to the reversible H<sup>+</sup>/H<sub>2</sub> value. <i>Biochem.</i> <b>38</b>: 8992-8999.</li>
	<li>Surerus, K.K., Chen, M., van der Zwaan, J.W., Rusnak, F.M., Kolk, M., Duin, E.C., Albracht, S.P., and Muenck, E. (1994) Further characterization of the spin coupling observed in oxidized hydrogenase from <i>Chromatium vinosum</i>. A M&ouml;ssbauer and multifrequency EPR study. <i>Biochem.</i> <b>33</b>: 4980-4993.</li>
	<li>Van der Zwaan, J.W., Albracht, S.P.J., Fontijn, R.D., and Slater, E.C. (1985) Monovalent nickel in hydrogenase from <i>Chromatium vinosum</i>. <i>FEBS Lett.</i> <b>179</b>: 271-277.</li>
	<li>Van der Zwaan, J.W., Albracht, S.P.J., Fontijn, R.D., and Roelofs, Y.B.M. (1986) EPR evidence for direct interaction of carbon monoxide with nickel in hydrogenase from <i>Chromatium vinosum</i>. <i>BBA Protein Struct. Mol. Enzymol.</i> <b>872</b>: 208-215.</li>
	<li>Van der Zwaan, J.W., Coremans, J.M.C.C., Bouwens, E.C.M., and Albracht, S.P.J. (1990) Effect of <sup>17</sup>O<sub>2</sub> and <sup>13</sup>CO on EPR espectra of nickel in hydrogenase from <i>Chromatium vinosum</i>. <i>BBA Protein Struct. Mol. Enzymol.</i> <b>1041:</b> 101-110.</li>
</ul>
	
	<h3>HydDB entries</h3>

	

			<footer class="text-center">
				<small class="text-muted">&copy; 2016 Dan S&oslash;ndergaard &amp; Chris Greening</small>
			</footer>
		</div>
	</body>
</html>

		